# translation of uk(5).po to Ukrainian
# Maxim Dziumanenko <mvd@mylinux.ua>, 2005.
# Vadim Abramchuck <Abram@email.ua>, 2006.
# Ukrainian translation of update-manager.
# Copyright (C) 2005, 2006 Free Software Foundation, Inc.
msgid ""
msgstr ""
"Project-Id-Version: uk(5)\n"
"Report-Msgid-Bugs-To: sebastian.heinlein@web.de\n"
"POT-Creation-Date: 2012-06-14 00:53+0100\n"
"PO-Revision-Date: 2012-04-13 01:02+0000\n"
"Last-Translator: Leonid Yaitskiy <Unknown>\n"
"Language-Team: Ukrainian <uk@li.org>\n"
"Language: uk\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=3; plural=n%10==1 && n%100!=11 ? 0 : n%10>=2 && n"
"%10<=4 && (n%100<10 || n%100>=20) ? 1 : 2;\n"
"X-Launchpad-Export-Date: 2012-04-17 13:41+0000\n"
"X-Generator: Launchpad (build 15099)\n"

#. TRANSLATORS: download size of small updates, e.g. "250 kB"
#: ../DistUpgrade/utils.py:433 ../UpdateManager/Core/utils.py:433
#, python-format
msgid "%(size).0f kB"
msgid_plural "%(size).0f kB"
msgstr[0] "%(size).0f кБ"
msgstr[1] "%(size).0f кБ"
msgstr[2] "%(size).0f кБ"

#. TRANSLATORS: download size of updates, e.g. "2.3 MB"
#: ../DistUpgrade/utils.py:436 ../UpdateManager/Core/utils.py:436
#, python-format
msgid "%.1f MB"
msgstr "%.1f Mб"

#. TRANSLATORS: %s is a country
#: ../DistUpgrade/distro.py:206 ../DistUpgrade/distro.py:436
#, python-format
msgid "Server for %s"
msgstr "Сервер для %s"

#. More than one server is used. Since we don't handle this case
#. in the user interface we set "custom servers" to true and
#. append a list of all used servers
#: ../DistUpgrade/distro.py:224 ../DistUpgrade/distro.py:230
#: ../DistUpgrade/distro.py:246
msgid "Main server"
msgstr "Основний сервер"

#: ../DistUpgrade/distro.py:250
msgid "Custom servers"
msgstr "Сервери користувача"

#: ../DistUpgrade/DistUpgradeAptCdrom.py:142
msgid "Could not calculate sources.list entry"
msgstr "Неможливо підрахувати запис sources.list"

#: ../DistUpgrade/DistUpgradeAptCdrom.py:251
msgid ""
"Unable to locate any package files, perhaps this is not a Ubuntu Disc or the "
"wrong architecture?"
msgstr ""
"Не вдалося знайти жодного файлу пакунків; можливо, це не диск Ubuntu або "
"диск для іншої архітектури?"

#: ../DistUpgrade/DistUpgradeAptCdrom.py:294
msgid "Failed to add the CD"
msgstr "Не вдалося додати компакт-диск"

#: ../DistUpgrade/DistUpgradeAptCdrom.py:295
#, python-format
msgid ""
"There was a error adding the CD, the upgrade will abort. Please report this "
"as a bug if this is a valid Ubuntu CD.\n"
"\n"
"The error message was:\n"
"'%s'"
msgstr ""
"Виникла помилка при додаванні компакт-диску, оновлення буде припинено. Будь-"
"ласка, повідомте про цю помилку, якщо це правильний компакт-диск Ubuntu.\n"
"Повідомлення про помилку:\n"
"'%s'"

#: ../DistUpgrade/DistUpgradeCache.py:151
msgid "Remove package in bad state"
msgid_plural "Remove packages in bad state"
msgstr[0] "Вилучити пакунок, що знаходиться в поганому стані"
msgstr[1] "Вилучити пакунки, що знаходяться в поганому стані"
msgstr[2] "Вилучити пакунки, що знаходяться в поганому стані"

#: ../DistUpgrade/DistUpgradeCache.py:154
#, python-format
msgid ""
"The package '%s' is in an inconsistent state and needs to be reinstalled, "
"but no archive can be found for it. Do you want to remove this package now "
"to continue?"
msgid_plural ""
"The packages '%s' are in an inconsistent state and need to be reinstalled, "
"but no archives can be found for them. Do you want to remove these packages "
"now to continue?"
msgstr[0] ""
"Пакунок '%s' знаходиться у несумісному стані і повинен бути "
"перевстановлений, але не знайдено жодного архіву для нього. Чи ви бажаєте "
"видалити цей пакунок зараз, щоб продовжити?"
msgstr[1] ""
"Пакунки'%s' знаходяться у несумісному стані і повинні бути перевстановлені, "
"але не знайдено жодного архіву для них. Чи ви бажаєте видалити ці пакунки "
"зараз, щоб продовжити?"
msgstr[2] ""
"Пакунки '%s' знаходяться у несумісному стані і повинні бути перевстановлені, "
"але не знайдено жодного архіву для них. Чи ви бажаєте видалити ці пакунки "
"зараз, щоб продовжити?"

#. FIXME: not ideal error message, but we just reuse a
#. existing one here to avoid a new string
#: ../DistUpgrade/DistUpgradeCache.py:255
msgid "The server may be overloaded"
msgstr "Схоже, що сервер перенавантажений"

#: ../DistUpgrade/DistUpgradeCache.py:368
msgid "Broken packages"
msgstr "Пошкоджені пакунки"

#: ../DistUpgrade/DistUpgradeCache.py:369
msgid ""
"Your system contains broken packages that couldn't be fixed with this "
"software. Please fix them first using synaptic or apt-get before proceeding."
msgstr ""
"Ваша система містить пошкоджені пакунки, котрі не можуть бути виправлені "
"цією програмою. Будь-ласка, виправіть їх програмами synaptic або apt-get."

#. FIXME: change the text to something more useful
#: ../DistUpgrade/DistUpgradeCache.py:693
#, python-format
msgid ""
"An unresolvable problem occurred while calculating the upgrade:\n"
"%s\n"
"\n"
" This can be caused by:\n"
" * Upgrading to a pre-release version of Ubuntu\n"
" * Running the current pre-release version of Ubuntu\n"
" * Unofficial software packages not provided by Ubuntu\n"
"\n"
msgstr ""
"Виникла невиправна помилка при розрахунку оновлення:\n"
"%s\n"
"\n"
" Причиною можуть бути:\n"
" * Оновлення Ubuntu до версії pre-release\n"
" * Використання поточної pre-release-версії Ubuntu\n"
" * Неофіційні пакунки програм, що не підтримуються Ubuntu\n"
"\n"

#: ../DistUpgrade/DistUpgradeCache.py:703
msgid "This is most likely a transient problem, please try again later."
msgstr "Перелік змін ще не готовий, спробуйте пізніше."

#: ../DistUpgrade/DistUpgradeCache.py:706
msgid ""
"If none of this applies, then please report this bug using the command "
"'ubuntu-bug update-manager' in a terminal."
msgstr ""
"Якщо нічого з цього не спрацює, тоді повідомте, будь-ласка, про цю помилку, "
"використовуючи в Терміналі команду 'ubuntu-bug update-manager'."

#: ../DistUpgrade/DistUpgradeCache.py:711
#: ../UpdateManager/UpdateManager.py:1031
msgid "Could not calculate the upgrade"
msgstr "Не вдалося розрахувати оновлення"

#: ../DistUpgrade/DistUpgradeCache.py:762
msgid "Error authenticating some packages"
msgstr "Помилка аутенфікації деяких пакетів"

#: ../DistUpgrade/DistUpgradeCache.py:763
msgid ""
"It was not possible to authenticate some packages. This may be a transient "
"network problem. You may want to try again later. See below for a list of "
"unauthenticated packages."
msgstr ""
"Не вдалося перевірити деякі пакети. Це може бути викликано проблемами в "
"мережі. Можливо, Ви захочете спробувати пізніше. Список не перевірених "
"пакетів надано нижче."

#: ../DistUpgrade/DistUpgradeCache.py:783
#, python-format
msgid ""
"The package '%s' is marked for removal but it is in the removal blacklist."
msgstr ""
"Пакунок '%s' помічено для видалення, але він в чорному списку видалення."

#: ../DistUpgrade/DistUpgradeCache.py:787
#, python-format
msgid "The essential package '%s' is marked for removal."
msgstr "Важливий пакунок '%s' помічено для видалення."

#: ../DistUpgrade/DistUpgradeCache.py:796
#, python-format
msgid "Trying to install blacklisted version '%s'"
msgstr "Спроба встановити версію '%s' чорного списку"

#: ../DistUpgrade/DistUpgradeCache.py:914
#, python-format
msgid "Can't install '%s'"
msgstr "Не можливо встановити '%s'"

#: ../DistUpgrade/DistUpgradeCache.py:915
msgid ""
"It was impossible to install a required package. Please report this as a bug "
"using 'ubuntu-bug update-manager' in a terminal."
msgstr ""
"Не вдалось встановити потрібний пакунок. Повідомте, будь-ласка, про цю "
"помилку, використовуючи в Терміналі команду 'ubuntu-bug update-manager'."

#. FIXME: provide a list
#: ../DistUpgrade/DistUpgradeCache.py:926
msgid "Can't guess meta-package"
msgstr "Не можливо підібрати meta-пакунок"

#: ../DistUpgrade/DistUpgradeCache.py:927
msgid ""
"Your system does not contain a ubuntu-desktop, kubuntu-desktop, xubuntu-"
"desktop or edubuntu-desktop package and it was not possible to detect which "
"version of Ubuntu you are running.\n"
" Please install one of the packages above first using synaptic or apt-get "
"before proceeding."
msgstr ""
"Ваша система не містить пакунків ubuntu-desktop, kubuntu-deskto, xubuntu-"
"desktop або edubuntu-desktop, через що не вдалося встановити, яку версію "
"Ubuntu ви використовуєте.\n"
" Будь ласка, спочатку встановіть один з цих пакунків, використовуючи "
"synaptic або apt-get."

#: ../DistUpgrade/DistUpgradeController.py:114
msgid "Reading cache"
msgstr "Зчитування кешу"

#: ../DistUpgrade/DistUpgradeController.py:223
msgid "Unable to get exclusive lock"
msgstr "Не вдалося отримати ексклюзивне блокування"

#: ../DistUpgrade/DistUpgradeController.py:224
msgid ""
"This usually means that another package management application (like apt-get "
"or aptitude) already running. Please close that application first."
msgstr ""
"Це зазвичай значить, що запущено інший менеджер пакунків (наприклад, apt-get "
"або aptitude). Будь ласка, спочатку закрийте цю програму."

#: ../DistUpgrade/DistUpgradeController.py:257
msgid "Upgrading over remote connection not supported"
msgstr "Віддалене оновлення системи більше не підтримується"

#: ../DistUpgrade/DistUpgradeController.py:258
msgid ""
"You are running the upgrade over a remote ssh connection with a frontend "
"that does not support this. Please try a text mode upgrade with 'do-release-"
"upgrade'.\n"
"\n"
"The upgrade will abort now. Please try without ssh."
msgstr ""
"Ви намагаєтесь виконати оновлення через ssh-з'єднання з непідтримуваним "
"клієнтом. Оновіться у текстовому режимі через \"do-release-upgrade\".\n"
"\n"
"Оновлення зупинене. Спробуйте без ssh."

#: ../DistUpgrade/DistUpgradeController.py:272
msgid "Continue running under SSH?"
msgstr "Продовжити роботу через SSH?"

#: ../DistUpgrade/DistUpgradeController.py:273
#, python-format
msgid ""
"This session appears to be running under ssh. It is not recommended to "
"perform a upgrade over ssh currently because in case of failure it is harder "
"to recover.\n"
"\n"
"If you continue, an additional ssh daemon will be started at port '%s'.\n"
"Do you want to continue?"
msgstr ""
"Ця сесія, скоріш за все, працює через ssh. Наполегливо рекомендуємо не "
"виконувати оновлення через ssh через те, що в разі невдачі буде важче "
"відновитись.\n"
"\n"
"Якщо Ви продовжите, буде запущено додаткову службу ssh на порті '%s'.\n"
"Чи бажаєте продовжити?"

#: ../DistUpgrade/DistUpgradeController.py:287
msgid "Starting additional sshd"
msgstr "Запуск додаткового sshd"

#: ../DistUpgrade/DistUpgradeController.py:288
#, python-format
msgid ""
"To make recovery in case of failure easier, an additional sshd will be "
"started on port '%s'. If anything goes wrong with the running ssh you can "
"still connect to the additional one.\n"
msgstr ""
"Щоб зробити відновлення легшим у разі поломки, на порті '%s' буде запущений "
"додатковий sshd. Якщо щось піде не так із запуском ssh, Ви все ще зможете "
"під'єднатись до другого.\n"

#: ../DistUpgrade/DistUpgradeController.py:296
#, python-format
msgid ""
"If you run a firewall, you may need to temporarily open this port. As this "
"is potentially dangerous it's not done automatically. You can open the port "
"with e.g.:\n"
"'%s'"
msgstr ""
"Якщо брандмауер увімкнений, треба тимчасово відкрити цей порт. Це потенційно "
"небезпечно, тому потребує втручання користувача. Ви можете відкрити порт "
"наступним чином:\n"
"'%s'"

#: ../DistUpgrade/DistUpgradeController.py:368
#: ../DistUpgrade/DistUpgradeController.py:413
msgid "Can not upgrade"
msgstr "Оновлення неможливе"

#: ../DistUpgrade/DistUpgradeController.py:369
#, python-format
msgid "An upgrade from '%s' to '%s' is not supported with this tool."
msgstr "Оновлення з '%s' до '%s' цією програмою не підтримується."

#: ../DistUpgrade/DistUpgradeController.py:378
msgid "Sandbox setup failed"
msgstr "Помилка встановлення в пісочниці"

#: ../DistUpgrade/DistUpgradeController.py:379
msgid "It was not possible to create the sandbox environment."
msgstr "Не вдалося створити середовище для пісочниці"

#: ../DistUpgrade/DistUpgradeController.py:385
msgid "Sandbox mode"
msgstr "Режим пісочниці"

#: ../DistUpgrade/DistUpgradeController.py:386
#, python-format
msgid ""
"This upgrade is running in sandbox (test) mode. All changes are written to "
"'%s' and will be lost on the next reboot.\n"
"\n"
"*No* changes written to a system directory from now until the next reboot "
"are permanent."
msgstr ""
"Це оновлення запускається у режимі пісочниці (випробовуваному). Всі зміни "
"записуються у '%s' і будуть втрачені при наступному перезапуску.\n"
"\n"
"*Ні* зміни записані до системної директорії з цього моменту до наступного "
"перезапуску будуть постійні."

#: ../DistUpgrade/DistUpgradeController.py:414
msgid ""
"Your python install is corrupted. Please fix the '/usr/bin/python' symlink."
msgstr ""
"Ваше встановлення пакету python пошкоджене. Виправте посилання '/usr/bin/"
"python'."

#: ../DistUpgrade/DistUpgradeController.py:440
msgid "Package 'debsig-verify' is installed"
msgstr "Пакунок 'debsig-verify' встановлено"

#: ../DistUpgrade/DistUpgradeController.py:441
msgid ""
"The upgrade can not continue with that package installed.\n"
"Please remove it with synaptic or 'apt-get remove debsig-verify' first and "
"run the upgrade again."
msgstr ""
"Оновлення не може бути продовжене, якщо цей пакунок встановлено.\n"
"Будь ласка, спочатку вилучіть його за домогою synaptic або командою 'apt-get "
"remove debsig-verify' та спробуйте запустити оновлення знову."

#: ../DistUpgrade/DistUpgradeController.py:453
#, python-format
msgid "Can not write to '%s'"
msgstr "Неможливо виконати запис в '%s'"

#: ../DistUpgrade/DistUpgradeController.py:454
#, python-format
msgid ""
"Its not possible to write to the system directory '%s' on your system. The "
"upgrade can not continue.\n"
"Please make sure that the system directory is writable."
msgstr ""
"Неможливо виконати запис в системний каталог '%s' у Вашій системі. Оновлення "
"не може бути продовженим.\n"
"Переконайтеся, що системний каталог доступний для запису."

#: ../DistUpgrade/DistUpgradeController.py:465
msgid "Include latest updates from the Internet?"
msgstr "Додати останні оновлення з Інтернету?"

#: ../DistUpgrade/DistUpgradeController.py:466
msgid ""
"The upgrade system can use the internet to automatically download the latest "
"updates and install them during the upgrade.  If you have a network "
"connection this is highly recommended.\n"
"\n"
"The upgrade will take longer, but when it is complete, your system will be "
"fully up to date.  You can choose not to do this, but you should install the "
"latest updates soon after upgrading.\n"
"If you answer 'no' here, the network is not used at all."
msgstr ""
"Система оновлення може використовувати Інтернет для автоматичного "
"завантаження останніх оновлень та встановлювати їх під час оновлення "
"системи. Якщо у вас є мережеве підключення — це наполегливо рекомендується.\n"
"\n"
"Оновлення буде більш тривалим, проте коли завершиться, Ваша система буде "
"повністю оновлена. Ви можете відмовитись від цього, але ви маєте встановити "
"останні оновлення одразу після завершення процесу оновлення.\n"
"Якщо Ви тут відповісте 'ні', мережа не буде використовуватись взагалі."

#: ../DistUpgrade/DistUpgradeController.py:686
#, python-format
msgid "disabled on upgrade to %s"
msgstr "відключено під час оновлення до %s"

#: ../DistUpgrade/DistUpgradeController.py:713
msgid "No valid mirror found"
msgstr "Не знайдено правильного дзеркала"

#: ../DistUpgrade/DistUpgradeController.py:714
#, python-format
msgid ""
"While scanning your repository information no mirror entry for the upgrade "
"was found. This can happen if you run a internal mirror or if the mirror "
"information is out of date.\n"
"\n"
"Do you want to rewrite your 'sources.list' file anyway? If you choose 'Yes' "
"here it will update all '%s' to '%s' entries.\n"
"If you select 'No' the upgrade will cancel."
msgstr ""
"Під час сканування інформації у репозиторію не було знайдено запису про "
"дзеркало для оновлення. Це може трапитися, якщо ви використовуєте внутрішнє "
"дзеркало або інформація про дзеркала застаріла.\n"
"\n"
"Ви все ж таки бажаєте перезаписати 'sources.list'? Якщо ви оберете 'Так', то "
"це оновить усі '%s' з '%s' записи.\n"
"Якщо ви оберете 'Ні' оновлення не відбудеться."

#. hm, still nothing useful ...
#: ../DistUpgrade/DistUpgradeController.py:734
msgid "Generate default sources?"
msgstr "Створити типові джерела?"

#: ../DistUpgrade/DistUpgradeController.py:735
#, python-format
msgid ""
"After scanning your 'sources.list' no valid entry for '%s' was found.\n"
"\n"
"Should default entries for '%s' be added? If you select 'No', the upgrade "
"will cancel."
msgstr ""
"Після сканування вашого 'sources.list' не було знайдено дійсного запису для "
"'%s'.\n"
"\n"
"Додати типовий запис для '%s'? Якщо ви оберете 'Ні', оновлення буде "
"скасовано."

#: ../DistUpgrade/DistUpgradeController.py:770
msgid "Repository information invalid"
msgstr "Помилка в даних про сховище"

#: ../DistUpgrade/DistUpgradeController.py:771
msgid ""
"Upgrading the repository information resulted in a invalid file so a bug "
"reporting process is being started."
msgstr ""
"Оновлення інформації про репозиторій закінчилося невдачею (недійсний файл), "
"тому запустився процес звітування про помилку."

#: ../DistUpgrade/DistUpgradeController.py:778
msgid "Third party sources disabled"
msgstr "Сторонні джерела відключені"

#: ../DistUpgrade/DistUpgradeController.py:779
msgid ""
"Some third party entries in your sources.list were disabled. You can re-"
"enable them after the upgrade with the 'software-properties' tool or your "
"package manager."
msgstr ""
"Деякі сторонні джерела у Вашому sources.list були відключені. Ви можете "
"знову включити їх після оновлення за допомогою програми 'software-"
"properties' чи менеджера пакунків."

#: ../DistUpgrade/DistUpgradeController.py:819
msgid "Package in inconsistent state"
msgid_plural "Packages in inconsistent state"
msgstr[0] "Пакунок у незадовільному стані"
msgstr[1] "Пакунки у незадовільному стані"
msgstr[2] "Пакунків у незадовільному стані"

#: ../DistUpgrade/DistUpgradeController.py:822
#, python-format
msgid ""
"The package '%s' is in an inconsistent state and needs to be reinstalled, "
"but no archive can be found for it. Please reinstall the package manually or "
"remove it from the system."
msgid_plural ""
"The packages '%s' are in an inconsistent state and need to be reinstalled, "
"but no archive can be found for them. Please reinstall the packages manually "
"or remove them from the system."
msgstr[0] ""
"Пакунок '%s' знаходиться в несумісному стані і повинен бути "
"перевстановлений, але не знайдено жодного архіву для нього. Будь ласка, "
"перевстановіть пакунок вручну чи вилучіть його з системи."
msgstr[1] ""
"Пакунки '%s' знаходяться в несумісному стані, було б добре їх "
"перевстановити, та не вдається знайти для них ніяких архівів. Будь ласка, "
"перевстановіть пакунки вручну чи вилучіть їх із системи."
msgstr[2] ""
"Пакунки '%s' знаходяться в несумісному стані і повинні бути перевстановлені, "
"але не знайдено жодного архіву для них. Будь ласка, перевстановіть пакунки "
"вручну чи вилучіть їх з системи."

#: ../DistUpgrade/DistUpgradeController.py:870
msgid "Error during update"
msgstr "Помилка під час оновлення"

#: ../DistUpgrade/DistUpgradeController.py:871
msgid ""
"A problem occurred during the update. This is usually some sort of network "
"problem, please check your network connection and retry."
msgstr ""
"Під час оновлення сталася помилка. Це зазвичай викликано проблемами у "
"мережі, будь ласка, перевірте Ваше мережеве з’єднання та спробуйте знову."

#. print("on_button_install_clicked")
#: ../DistUpgrade/DistUpgradeController.py:880
#: ../UpdateManager/UpdateManager.py:757
msgid "Not enough free disk space"
msgstr "Недостатньо місця на диску"

#: ../DistUpgrade/DistUpgradeController.py:881
#, python-format
msgid ""
"The upgrade has aborted. The upgrade needs a total of %s free space on disk "
"'%s'. Please free at least an additional %s of disk space on '%s'. Empty "
"your trash and remove temporary packages of former installations using 'sudo "
"apt-get clean'."
msgstr ""
"Оновлення було перервано. Для оновлення необхідно %s вільного місця на диску "
"'%s'. Будь ласка, звільніть додатково хоча б %s дискового простору на '%s'. "
"Очистіть кошик та вилучіть тимчасові пакунки з попередніх установок, "
"використовуючи 'sudo apt-get clean'."

#. calc the dist-upgrade and see if the removals are ok/expected
#. do the dist-upgrade
#: ../DistUpgrade/DistUpgradeController.py:910
#: ../DistUpgrade/DistUpgradeController.py:1692
msgid "Calculating the changes"
msgstr "Проводиться аналіз змін"

#. ask the user
#: ../DistUpgrade/DistUpgradeController.py:942
msgid "Do you want to start the upgrade?"
msgstr "Бажаєте почати оновлення системи?"

#: ../DistUpgrade/DistUpgradeController.py:1008
msgid "Upgrade canceled"
msgstr "Оновлення скасовано"

#: ../DistUpgrade/DistUpgradeController.py:1009
msgid ""
"The upgrade will cancel now and the original system state will be restored. "
"You can resume the upgrade at a later time."
msgstr ""
"Оновлення буде скасовано та систему буде відновлено до попереднього стану. "
"Ви зможете продовжити оновлення пізніше."

#: ../DistUpgrade/DistUpgradeController.py:1015
#: ../DistUpgrade/DistUpgradeController.py:1149
msgid "Could not download the upgrades"
msgstr "Неможливо завантажити пакунки для оновлення системи"

#: ../DistUpgrade/DistUpgradeController.py:1016
msgid ""
"The upgrade has aborted. Please check your Internet connection or "
"installation media and try again. All files downloaded so far have been kept."
msgstr ""
"Оновлення було перервано. Будь ласка, перевірте ваше інтернет-з'єднання чи "
"носій, з якого здійснюється встановлення, та спробуйте знову. Всі "
"завантажені файли будуть збережені."

#. FIXME: strings are not good, but we are in string freeze
#. currently
#: ../DistUpgrade/DistUpgradeController.py:1100
#: ../DistUpgrade/DistUpgradeController.py:1137
#: ../DistUpgrade/DistUpgradeController.py:1242
msgid "Error during commit"
msgstr "Помилка при фіксації"

#. generate a new cache
#: ../DistUpgrade/DistUpgradeController.py:1102
#: ../DistUpgrade/DistUpgradeController.py:1139
#: ../DistUpgrade/DistUpgradeController.py:1281
msgid "Restoring original system state"
msgstr "Відновлення первісного стану системи"

#: ../DistUpgrade/DistUpgradeController.py:1103
#: ../DistUpgrade/DistUpgradeController.py:1118
#: ../DistUpgrade/DistUpgradeController.py:1140
msgid "Could not install the upgrades"
msgstr "Неможливо провести оновлення системи"

#. invoke the frontend now and show a error message
#: ../DistUpgrade/DistUpgradeController.py:1108
msgid ""
"The upgrade has aborted. Your system could be in an unusable state. A "
"recovery will run now (dpkg --configure -a)."
msgstr ""
"Оновлення було перервано. Ваша система може бути в нестабільному стані. "
"Зараз буде запущено процес відновлення (dpkg --configure -a)."

#: ../DistUpgrade/DistUpgradeController.py:1113
#, python-format
msgid ""
"\n"
"\n"
"Please report this bug in a browser at http://bugs.launchpad.net/ubuntu/"
"+source/update-manager/+filebug and attach the files in /var/log/dist-"
"upgrade/ to the bug report.\n"
"%s"
msgstr ""
"\n"
"\n"
"Будь ласка, повідомте про цю помилку в браузері на http://bugs.launchpad.net/"
"ubuntu/+source/update-manager/+filebug і прикріпіть файли в /var/log/dist-"
"upgrade/ до звіту.\n"
"%s"

#: ../DistUpgrade/DistUpgradeController.py:1150
msgid ""
"The upgrade has aborted. Please check your Internet connection or "
"installation media and try again. "
msgstr ""
"Оновлення було перервано. Будь ласка, перевірте Ваше з'єднання з Інтернетом "
"чи носій для встановлення та спробуйте знову. "

#: ../DistUpgrade/DistUpgradeController.py:1230
msgid "Remove obsolete packages?"
msgstr "Видалити застарілі пакунки?"

#: ../DistUpgrade/DistUpgradeController.py:1231
#: ../DistUpgrade/DistUpgrade.ui.h:8
msgid "_Keep"
msgstr "_Залишити"

#: ../DistUpgrade/DistUpgradeController.py:1231
msgid "_Remove"
msgstr "Видалити"

#: ../DistUpgrade/DistUpgradeController.py:1243
msgid ""
"A problem occurred during the clean-up. Please see the below message for "
"more information. "
msgstr ""
"При очищенні системи виникли проблеми. Прочитайте детальнішу інформацію "
"нижче. "

#. FIXME: instead of error out, fetch and install it
#. here
#: ../DistUpgrade/DistUpgradeController.py:1319
msgid "Required depends is not installed"
msgstr "Необхідний пакет (залежність) не встановлений"

#: ../DistUpgrade/DistUpgradeController.py:1320
#, python-format
msgid "The required dependency '%s' is not installed. "
msgstr "Необхідний пакет (залежність) '%s' не встановлений. "

#. sanity check (check for ubuntu-desktop, brokenCache etc)
#. then open the cache (again)
#: ../DistUpgrade/DistUpgradeController.py:1588
#: ../DistUpgrade/DistUpgradeController.py:1653
msgid "Checking package manager"
msgstr "Перевірка програми управління пакунками"

#: ../DistUpgrade/DistUpgradeController.py:1593
msgid "Preparing the upgrade failed"
msgstr "Підготовка до оновлення не вдалася"

#: ../DistUpgrade/DistUpgradeController.py:1594
msgid ""
"Preparing the system for the upgrade failed so a bug reporting process is "
"being started."
msgstr ""
"Підготовка системи до оновлення закінчилася невдало. Почато процес "
"звітування про помилку."

#: ../DistUpgrade/DistUpgradeController.py:1608
msgid "Getting upgrade prerequisites failed"
msgstr "Помилка отримання необхідних файлів для оновлення"

#: ../DistUpgrade/DistUpgradeController.py:1609
msgid ""
"The system was unable to get the prerequisites for the upgrade. The upgrade "
"will abort now and restore the original system state.\n"
"\n"
"Additionally, a bug reporting process is being started."
msgstr ""
"Системі не вдалося отримати попередні конфігурації для повного оновлення. "
"Зараз процес оновлення буде перервано і буде відновлено початковий стан "
"системи.\n"
"\n"
"Також почато процес звітування про помилку."

#: ../DistUpgrade/DistUpgradeController.py:1637
msgid "Updating repository information"
msgstr "Оновлення інформації про сховище"

#: ../DistUpgrade/DistUpgradeController.py:1644
msgid "Failed to add the cdrom"
msgstr "Помилка при підключенні CD-ROM"

#: ../DistUpgrade/DistUpgradeController.py:1645
msgid "Sorry, adding the cdrom was not successful."
msgstr "На жаль, не вдалося підключити CD-ROM"

#: ../DistUpgrade/DistUpgradeController.py:1673
msgid "Invalid package information"
msgstr "Невірна інформація про пакунок"

#: ../DistUpgrade/DistUpgradeController.py:1674
msgid "After updating your package "
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:1698
#: ../DistUpgrade/DistUpgradeController.py:1750
msgid "Fetching"
msgstr "Отримання"

#: ../DistUpgrade/DistUpgradeController.py:1704
#: ../DistUpgrade/DistUpgradeController.py:1754
msgid "Upgrading"
msgstr "Процес оновлення"

#. don't abort here, because it would restore the sources.list
#: ../DistUpgrade/DistUpgradeController.py:1709
#: ../DistUpgrade/DistUpgradeController.py:1756
#: ../DistUpgrade/DistUpgradeController.py:1763
#: ../DistUpgrade/DistUpgradeController.py:1774
msgid "Upgrade complete"
msgstr "Оновлення завершено"

#: ../DistUpgrade/DistUpgradeController.py:1710
#: ../DistUpgrade/DistUpgradeController.py:1757
#: ../DistUpgrade/DistUpgradeController.py:1764
msgid ""
"The upgrade has completed but there were errors during the upgrade process."
msgstr "Оновлення було завершено, але в його процесі виникли деякі помилки."

#: ../DistUpgrade/DistUpgradeController.py:1717
msgid "Searching for obsolete software"
msgstr "Пошук застарілих програм"

#: ../DistUpgrade/DistUpgradeController.py:1726
msgid "System upgrade is complete."
msgstr "Оновлення системи завершено."

#: ../DistUpgrade/DistUpgradeController.py:1775
msgid "The partial upgrade was completed."
msgstr "Часткове оновлення завершене."

#: ../DistUpgrade/DistUpgradeQuirks.py:204
msgid "evms in use"
msgstr "використовується evms"

#: ../DistUpgrade/DistUpgradeQuirks.py:205
msgid ""
"Your system uses the 'evms' volume manager in /proc/mounts. The 'evms' "
"software is no longer supported, please switch it off and run the upgrade "
"again when this is done."
msgstr ""
"Ваша система використовує менеджер томів 'evms' в /proc/mounts. Програма "
"'evms' більше не підтримується, будь ласка, вимкніть її та запустіть "
"оновлення знову."

#: ../DistUpgrade/DistUpgradeQuirks.py:502
msgid "Your graphics hardware may not be fully supported in Ubuntu 12.04 LTS."
msgstr "Ваш графічний пристрій підтримується Ubuntu 12.04 LTS не повністю."

#: ../DistUpgrade/DistUpgradeQuirks.py:504
msgid ""
"The support in Ubuntu 12.04 LTS for your Intel graphics hardware is limited "
"and you may encounter problems after the upgrade. For more information see "
"https://wiki.ubuntu.com/X/Bugs/UpdateManagerWarningForI8xx Do you want to "
"continue with the upgrade?"
msgstr ""
"Підтримку Вашого графічного адаптера Intel в Ubuntu 12.04 LTS обмежено, і Ви "
"можете стикнутися з проблемами після оновлення. Для отримання детальної "
"інформації відвідайте https://wiki.ubuntu.com/X/Bugs/"
"UpdateManagerWarningForI8xx Ви дійсно бажаєте продовжити оновлення?"

#: ../DistUpgrade/DistUpgradeQuirks.py:526
#: ../DistUpgrade/DistUpgradeQuirks.py:554
#: ../DistUpgrade/DistUpgradeQuirks.py:581
msgid ""
"Upgrading may reduce desktop effects, and performance in games and other "
"graphically intensive programs."
msgstr ""
"Оновлення може уповільнити ефекти робочого столу, продуктивність роботи в "
"іграх та інших графічно складних програмах."

#: ../DistUpgrade/DistUpgradeQuirks.py:530
#: ../DistUpgrade/DistUpgradeQuirks.py:558
msgid ""
"This computer is currently using the NVIDIA 'nvidia' graphics driver. No "
"version of this driver is available that works with your video card in "
"Ubuntu 10.04 LTS.\n"
"\n"
"Do you want to continue?"
msgstr ""
"Цей комп'ютер зараз використовує графічний драйвер NVIDIA 'nvidia'. Нема "
"доступних версій драйверів, які б працювали з вашим обладнанням на Ubuntu "
"10.04 LTS.\n"
"\n"
"Чи хочете ви продовжувати?"

#: ../DistUpgrade/DistUpgradeQuirks.py:585
msgid ""
"This computer is currently using the AMD 'fglrx' graphics driver. No version "
"of this driver is available that works with your hardware in Ubuntu 10.04 "
"LTS.\n"
"\n"
"Do you want to continue?"
msgstr ""
"Цей комп'ютер зараз використовує графічний драйвер AMD 'fglrx'. Нема "
"доступних версій драйверів, які б працювали з вашим обладнанням на Ubuntu "
"10.04 LTS.\n"
"\n"
"Чи хочете ви продовжувати?"

#: ../DistUpgrade/DistUpgradeQuirks.py:615
msgid "No i686 CPU"
msgstr "Немає процесора i686"

#: ../DistUpgrade/DistUpgradeQuirks.py:616
msgid ""
"Your system uses an i586 CPU or a CPU that does not have the 'cmov' "
"extension. All packages were built with optimizations requiring i686 as the "
"minimal architecture. It is not possible to upgrade your system to a new "
"Ubuntu release with this hardware."
msgstr ""
"У вашому комп’ютері стоїть процесор i586 або інший, що не має розширення "
"'cmov'. Усі ж пакети було зібрано з оптимізацією, що вимагає мінімум i686-"
"архітектуру. Неможливо оновити систему до нової версії Ubuntu на цьому "
"комп’ютері."

#: ../DistUpgrade/DistUpgradeQuirks.py:652
msgid "No ARMv6 CPU"
msgstr "Немає ARMv6 CPU"

#: ../DistUpgrade/DistUpgradeQuirks.py:653
msgid ""
"Your system uses an ARM CPU that is older than the ARMv6 architecture. All "
"packages in karmic were built with optimizations requiring ARMv6 as the "
"minimal architecture. It is not possible to upgrade your system to a new "
"Ubuntu release with this hardware."
msgstr ""
"Ваша система використовує процесор ARM з архітектурою, старішою за ARMv6. "
"Всі пакети у karmic були скомпільовані з оптимізаціями, які вимагають ARMv6 "
"як мінімум. Оновлення вашої системи до нового випуску Ubuntu неможливе на "
"даному обладнанні."

#: ../DistUpgrade/DistUpgradeQuirks.py:673
msgid "No init available"
msgstr "Служба init недоступна"

#: ../DistUpgrade/DistUpgradeQuirks.py:674
msgid ""
"Your system appears to be a virtualised environment without an init daemon, "
"e.g. Linux-VServer. Ubuntu 10.04 LTS cannot function within this type of "
"environment, requiring an update to your virtual machine configuration "
"first.\n"
"\n"
"Are you sure you want to continue?"
msgstr ""
"Схоже, що ваша система є віртуалізованим оточенням без служби init, "
"наприклад Linux-VServer. Ubuntu 10.04 LTS не може працювати в такому типі "
"оточення, ви повинні спочатку виправити конфігурацію своєї віртуальної "
"машини.\n"
"\n"
"Ви впевнені, що хочете продовжити?"

#: ../DistUpgrade/DistUpgradeMain.py:65
msgid "Sandbox upgrade using aufs"
msgstr "Оновлення Sandbox з використанням aufs"

#: ../DistUpgrade/DistUpgradeMain.py:67
msgid "Use the given path to search for a cdrom with upgradable packages"
msgstr "Вживати наступний шлях для пошуку компакт-диска з пакунками оновлення"

#: ../DistUpgrade/DistUpgradeMain.py:73
msgid ""
"Use frontend. Currently available: \n"
"DistUpgradeViewText, DistUpgradeViewGtk, DistUpgradeViewKDE"
msgstr ""
"Використовувати інтерфейс. В наявності: \n"
"DistUpgradeViewText, DistUpgradeViewGtk, DistUpgradeViewKDE"

#: ../DistUpgrade/DistUpgradeMain.py:76
msgid "*DEPRECATED* this option will be ignored"
msgstr "*ЗАСТАРІЛЕ* Цю опцію буде проігноровано"

#: ../DistUpgrade/DistUpgradeMain.py:79
msgid "Perform a partial upgrade only (no sources.list rewriting)"
msgstr "Виконати лише часткове оновлення (без перезапису sources.list)"

#: ../DistUpgrade/DistUpgradeMain.py:82
msgid "Disable GNU screen support"
msgstr "Вимкнути підтримку GNU screen"

#: ../DistUpgrade/DistUpgradeMain.py:84
msgid "Set datadir"
msgstr "Вказати каталог з даними"

#. print("mediaChange %s %s" % (medium, drive))
#: ../DistUpgrade/DistUpgradeViewGtk.py:114
#: ../DistUpgrade/DistUpgradeViewGtk3.py:117
#: ../DistUpgrade/DistUpgradeViewKDE.py:195
#: ../UpdateManager/DistUpgradeFetcherKDE.py:155
#, python-format
msgid "Please insert '%s' into the drive '%s'"
msgstr "Вставте '%s' в привід '%s'"

#: ../DistUpgrade/DistUpgradeViewGtk.py:135
#: ../DistUpgrade/DistUpgradeViewGtk3.py:138
#: ../DistUpgrade/DistUpgradeViewKDE.py:209
msgid "Fetching is complete"
msgstr "Отримання завершено"

#: ../DistUpgrade/DistUpgradeViewGtk.py:146
#: ../DistUpgrade/DistUpgradeViewGtk3.py:149
#: ../DistUpgrade/DistUpgradeViewKDE.py:222
#, python-format
msgid "Fetching file %li of %li at %sB/s"
msgstr "Отримання файлу %li з %li на швидкості %sB/s"

#: ../DistUpgrade/DistUpgradeViewGtk.py:149
#: ../DistUpgrade/DistUpgradeViewGtk.py:296
#: ../DistUpgrade/DistUpgradeViewGtk3.py:152
#: ../DistUpgrade/DistUpgradeViewGtk3.py:309
#: ../DistUpgrade/DistUpgradeViewKDE.py:223
#: ../DistUpgrade/DistUpgradeViewKDE.py:371
#, python-format
msgid "About %s remaining"
msgstr "Приблизно '%s' залишилось"

#: ../DistUpgrade/DistUpgradeViewGtk.py:152
#: ../DistUpgrade/DistUpgradeViewGtk3.py:155
#: ../DistUpgrade/DistUpgradeViewKDE.py:225
#, python-format
msgid "Fetching file %li of %li"
msgstr "Отримання файлу %li з %li"

#. FIXME: add support for the timeout
#. of the terminal (to display something useful then)
#. -> longer term, move this code into python-apt
#: ../DistUpgrade/DistUpgradeViewGtk.py:183
#: ../DistUpgrade/DistUpgradeViewGtk3.py:186
#: ../DistUpgrade/DistUpgradeViewKDE.py:262
msgid "Applying changes"
msgstr "Застосування змін"

#. we do not report followup errors from earlier failures
#: ../DistUpgrade/DistUpgradeViewGtk.py:208
#: ../DistUpgrade/DistUpgradeViewGtk3.py:212
#: ../DistUpgrade/DistUpgradeViewKDE.py:275
msgid "dependency problems - leaving unconfigured"
msgstr "проблеми з залежностями - залишається неналаштованим"

#: ../DistUpgrade/DistUpgradeViewGtk.py:213
#: ../DistUpgrade/DistUpgradeViewGtk3.py:217
#: ../DistUpgrade/DistUpgradeViewKDE.py:277
#, python-format
msgid "Could not install '%s'"
msgstr "Не вдалося встановити '%s'"

#: ../DistUpgrade/DistUpgradeViewGtk.py:214
#: ../DistUpgrade/DistUpgradeViewGtk3.py:218
#: ../DistUpgrade/DistUpgradeViewKDE.py:278
#, python-format
msgid ""
"The upgrade will continue but the '%s' package may not be in a working "
"state. Please consider submitting a bug report about it."
msgstr ""
"Обновлення продовжиться, але пакет '%s' може бути в неробочому стані. Будь "
"ласка, надішліть звіт про помилку."

#. self.expander.set_expanded(True)
#: ../DistUpgrade/DistUpgradeViewGtk.py:231
#: ../DistUpgrade/DistUpgradeViewGtk3.py:235
#: ../DistUpgrade/DistUpgradeViewKDE.py:299
#, python-format
msgid ""
"Replace the customized configuration file\n"
"'%s'?"
msgstr ""
"Замінити змінений користувачем конфігураційний файл\n"
"'%s'?"

#: ../DistUpgrade/DistUpgradeViewGtk.py:232
#: ../DistUpgrade/DistUpgradeViewGtk3.py:236
#: ../DistUpgrade/DistUpgradeViewKDE.py:300
msgid ""
"You will lose any changes you have made to this configuration file if you "
"choose to replace it with a newer version."
msgstr ""
"Ви втратите всі зроблені Вами у цьому конфігураційному файлі зміни якщо "
"заміните його новою версією."

#: ../DistUpgrade/DistUpgradeViewGtk.py:251
#: ../DistUpgrade/DistUpgradeViewGtk3.py:256
#: ../DistUpgrade/DistUpgradeViewKDE.py:323
msgid "The 'diff' command was not found"
msgstr "Команда 'diff' не знайдена"

#: ../DistUpgrade/DistUpgradeViewGtk.py:464
#: ../DistUpgrade/DistUpgradeViewGtk3.py:477
#: ../DistUpgrade/DistUpgradeViewText.py:92
msgid "A fatal error occurred"
msgstr "Виникла невиправна помилка"

#: ../DistUpgrade/DistUpgradeViewGtk.py:465
#: ../DistUpgrade/DistUpgradeViewGtk3.py:478
msgid ""
"Please report this as a bug (if you haven't already) and include the files /"
"var/log/dist-upgrade/main.log and /var/log/dist-upgrade/apt.log in your "
"report. The upgrade has aborted.\n"
"Your original sources.list was saved in /etc/apt/sources.list.distUpgrade."
msgstr ""
"Будь ласка, повідомте про цю помилку (якщо ви ще це не зробили) та додайте "
"фали /var/log/dist-upgrade/main.log та /var/log/dist-upgrade/apt.log до "
"вашого звіту. Оновлення було перервано.\n"
"Ваш оригінальний sources.list було збережено в /etc/apt/sources.list."
"distUpgrade."

#: ../DistUpgrade/DistUpgradeViewGtk.py:482
#: ../DistUpgrade/DistUpgradeViewGtk3.py:495
msgid "Ctrl-c pressed"
msgstr "Натиснено Ctrl-c"

#: ../DistUpgrade/DistUpgradeViewGtk.py:483
#: ../DistUpgrade/DistUpgradeViewGtk3.py:496
msgid ""
"This will abort the operation and may leave the system in a broken state. "
"Are you sure you want to do that?"
msgstr ""
"Операція буде скасована, що може залишити систему в несправному стані. Ви "
"все ще хочете це зробити?"

#. append warning
#: ../DistUpgrade/DistUpgradeViewGtk.py:631
#: ../DistUpgrade/DistUpgradeViewGtk3.py:629
msgid "To prevent data loss close all open applications and documents."
msgstr "Для запобігання втрати інформації закрийте усі програми та документи."

#: ../DistUpgrade/DistUpgradeViewGtk.py:645
#: ../DistUpgrade/DistUpgradeViewGtk3.py:643
#, python-format
msgid "No longer supported by Canonical (%s)"
msgstr "Більше не підтримується Canonical (%s)"

#: ../DistUpgrade/DistUpgradeViewGtk.py:646
#: ../DistUpgrade/DistUpgradeViewGtk3.py:644
#, python-format
msgid "<b>Downgrade (%s)</b>"
msgstr "<b>Встановлення старої версії (%s)</b>"

#: ../DistUpgrade/DistUpgradeViewGtk.py:647
#: ../DistUpgrade/DistUpgradeViewGtk3.py:645
#, python-format
msgid "Remove (%s)"
msgstr "Видалити (%s)"

#: ../DistUpgrade/DistUpgradeViewGtk.py:648
#: ../DistUpgrade/DistUpgradeViewGtk3.py:646
#, python-format
msgid "No longer needed (%s)"
msgstr "Більш не потрібен (%s)"

#: ../DistUpgrade/DistUpgradeViewGtk.py:649
#: ../DistUpgrade/DistUpgradeViewGtk3.py:647
#, python-format
msgid "Install (%s)"
msgstr "Встановлення (%s)"

#: ../DistUpgrade/DistUpgradeViewGtk.py:650
#: ../DistUpgrade/DistUpgradeViewGtk3.py:648
#, python-format
msgid "Upgrade (%s)"
msgstr "Оновлення (%s)"

#. change = QMessageBox.question(None, _("Media Change"), msg, QMessageBox.Ok, QMessageBox.Cancel)
#: ../DistUpgrade/DistUpgradeViewKDE.py:196
#: ../UpdateManager/DistUpgradeFetcherKDE.py:157
msgid "Media Change"
msgstr "Зміна носія"

#: ../DistUpgrade/DistUpgradeViewKDE.py:335
msgid "Show Difference >>>"
msgstr "Показати відмінності >>>"

#: ../DistUpgrade/DistUpgradeViewKDE.py:338
msgid "<<< Hide Difference"
msgstr "<<< Сховати відмінності"

#: ../DistUpgrade/DistUpgradeViewKDE.py:554
msgid "Error"
msgstr "Помилка"

#: ../DistUpgrade/DistUpgradeViewKDE.py:568
msgid "&Cancel"
msgstr "&Скасувати"

#: ../DistUpgrade/DistUpgradeViewKDE.py:572
#: ../DistUpgrade/DistUpgradeViewKDE.py:813
msgid "&Close"
msgstr "&Закрити"

#: ../DistUpgrade/DistUpgradeViewKDE.py:618
msgid "Show Terminal >>>"
msgstr "Показати Термінал >>>"

#: ../DistUpgrade/DistUpgradeViewKDE.py:621
msgid "<<< Hide Terminal"
msgstr "<<< Сховати Термінал"

#: ../DistUpgrade/DistUpgradeViewKDE.py:701
msgid "Information"
msgstr "Інформація"

#: ../DistUpgrade/DistUpgradeViewKDE.py:751
#: ../DistUpgrade/DistUpgradeViewKDE.py:796
#: ../DistUpgrade/DistUpgradeViewKDE.py:799 ../DistUpgrade/DistUpgrade.ui.h:7
msgid "Details"
msgstr "Деталі"

#: ../DistUpgrade/DistUpgradeViewKDE.py:777
#, python-format
msgid "No longer supported %s"
msgstr "Більш не підтримується %s"

#: ../DistUpgrade/DistUpgradeViewKDE.py:779
#, python-format
msgid "Remove %s"
msgstr "Видалити %s"

#: ../DistUpgrade/DistUpgradeViewKDE.py:781
#: ../DistUpgrade/DistUpgradeViewText.py:182
#, python-format
msgid "Remove (was auto installed) %s"
msgstr "Видалити (був автоматично інстальований) %s"

#: ../DistUpgrade/DistUpgradeViewKDE.py:783
#, python-format
msgid "Install %s"
msgstr "Встановити %s"

#: ../DistUpgrade/DistUpgradeViewKDE.py:785
#, python-format
msgid "Upgrade %s"
msgstr "Оновити %s"

#: ../DistUpgrade/DistUpgradeViewKDE.py:809
#: ../DistUpgrade/DistUpgradeViewText.py:230
msgid "Restart required"
msgstr "Необхідне перезавантаження"

#: ../DistUpgrade/DistUpgradeViewKDE.py:809
msgid "<b><big>Restart the system to complete the upgrade</big></b>"
msgstr "<b><big>Перезавантажте систему для завершення оновлення</big></b>"

#: ../DistUpgrade/DistUpgradeViewKDE.py:812 ../DistUpgrade/DistUpgrade.ui.h:14
#: ../data/gtkbuilder/UpdateManager.ui.h:26
msgid "_Restart Now"
msgstr "Перезапустити зараз"

#. FIXME make this user friendly
#: ../DistUpgrade/DistUpgradeViewKDE.py:830
msgid ""
"<b><big>Cancel the running upgrade?</big></b>\n"
"\n"
"The system could be in an unusable state if you cancel the upgrade. You are "
"strongly advised to resume the upgrade."
msgstr ""
"<b><big>Скасувати запущене оновлення?</big></b>\n"
"\n"
"Система може потрапити у непридатний стан, якщо скасувати оновлення. Вам "
"наполегливо рекомендується продовжити оновлення."

#: ../DistUpgrade/DistUpgradeViewKDE.py:834
msgid "Cancel Upgrade?"
msgstr "Скасувати оновлення?"

#: ../DistUpgrade/DistUpgradeView.py:61
#, python-format
msgid "%li day"
msgid_plural "%li days"
msgstr[0] "%li день"
msgstr[1] "%li дні"
msgstr[2] "%li днів"

#: ../DistUpgrade/DistUpgradeView.py:63
#, python-format
msgid "%li hour"
msgid_plural "%li hours"
msgstr[0] "%li година"
msgstr[1] "%li години"
msgstr[2] "%li годин"

#: ../DistUpgrade/DistUpgradeView.py:65
#, python-format
msgid "%li minute"
msgid_plural "%li minutes"
msgstr[0] "%li хвилина"
msgstr[1] "%li хвилини"
msgstr[2] "%li хвилин"

#: ../DistUpgrade/DistUpgradeView.py:66
#, python-format
msgid "%li second"
msgid_plural "%li seconds"
msgstr[0] "%li секунда"
msgstr[1] "%li секунди"
msgstr[2] "%li секунд"

#. TRANSLATORS: you can alter the ordering of the remaining time
#. information here if you shuffle %(str_days)s %(str_hours)s %(str_minutes)s
#. around. Make sure to keep all '$(str_*)s' in the translated string
#. and do NOT change anything appart from the ordering.
#.
#. %(str_hours)s will be either "1 hour" or "2 hours" depending on the
#. plural form
#.
#. Note: most western languages will not need to change this
#: ../DistUpgrade/DistUpgradeView.py:82
#, python-format
msgid "%(str_days)s %(str_hours)s"
msgstr "%(str_days)s %(str_hours)s"

#. TRANSLATORS: you can alter the ordering of the remaining time
#. information here if you shuffle %(str_hours)s %(str_minutes)s
#. around. Make sure to keep all '$(str_*)s' in the translated string
#. and do NOT change anything appart from the ordering.
#.
#. %(str_hours)s will be either "1 hour" or "2 hours" depending on the
#. plural form
#.
#. Note: most western languages will not need to change this
#: ../DistUpgrade/DistUpgradeView.py:100
#, python-format
msgid "%(str_hours)s %(str_minutes)s"
msgstr "%(str_hours)s %(str_minutes)s"

#. 56 kbit
#. 1Mbit = 1024 kbit
#: ../DistUpgrade/DistUpgradeView.py:151
#, python-format
msgid ""
"This download will take about %s with a 1Mbit DSL connection and about %s "
"with a 56k modem."
msgstr ""
"Це завантаження триватиме близько %s з 1Mбіт DSL-з'єднанням та біля %s з 56k "
"модемом."

#. if we have a estimated speed, use it
#: ../DistUpgrade/DistUpgradeView.py:155
#, python-format
msgid "This download will take about %s with your connection. "
msgstr "Це завантаження триватиме приблизно %s. "

#. Declare these translatable strings from the .ui files here so that
#. xgettext picks them up.
#: ../DistUpgrade/DistUpgradeView.py:259 ../DistUpgrade/DistUpgrade.ui.h:21
msgid "Preparing to upgrade"
msgstr "Підготовка до оновлення"

#: ../DistUpgrade/DistUpgradeView.py:260
msgid "Getting new software channels"
msgstr "Отримання нових каналів програм"

#: ../DistUpgrade/DistUpgradeView.py:261 ../DistUpgrade/DistUpgrade.ui.h:23
msgid "Getting new packages"
msgstr "Отримання нових пакунків"

#: ../DistUpgrade/DistUpgradeView.py:262 ../DistUpgrade/DistUpgrade.ui.h:26
msgid "Installing the upgrades"
msgstr "Встановлення оновлень"

#: ../DistUpgrade/DistUpgradeView.py:263 ../DistUpgrade/DistUpgrade.ui.h:25
msgid "Cleaning up"
msgstr "Очищення"

#: ../DistUpgrade/DistUpgradeView.py:348
#, python-format
msgid ""
"%(amount)d installed package is no longer supported by Canonical. You can "
"still get support from the community."
msgid_plural ""
"%(amount)d installed packages are no longer supported by Canonical. You can "
"still get support from the community."
msgstr[0] ""
"%(amount)d установлений пакет більш не підтримується Canonical. Ви можете "
"отримати підтримку у спільноти."
msgstr[1] ""
"%(amount)d установлених пакетів більш не підтримуються Canonical. Ви можете "
"отримати підтримку у спільноти."
msgstr[2] ""
"%(amount)d установлених пакетів більш не підтримуються Canonical. Ви можете "
"отримати підтримку у спільноти."

#. FIXME: make those two separate lines to make it clear
#. that the "%" applies to the result of ngettext
#: ../DistUpgrade/DistUpgradeView.py:357
#, python-format
msgid "%d package is going to be removed."
msgid_plural "%d packages are going to be removed."
msgstr[0] ""
"Copy text   \t\r\n"
"%d пакет буде видалено."
msgstr[1] "%d пакети буде видалено."
msgstr[2] "%d пакетів буде видалено."

#: ../DistUpgrade/DistUpgradeView.py:362
#, python-format
msgid "%d new package is going to be installed."
msgid_plural "%d new packages are going to be installed."
msgstr[0] "Буде встановлено %d новий пакет."
msgstr[1] "Буде встановлено %d нових пекети."
msgstr[2] "Буде встановлено %d нових пекетів."

#: ../DistUpgrade/DistUpgradeView.py:368
#, python-format
msgid "%d package is going to be upgraded."
msgid_plural "%d packages are going to be upgraded."
msgstr[0] "Буде оновлено %d пакунок."
msgstr[1] "Буде оновлено %d пакунки."
msgstr[2] "Буде оновлено %d пакунків."

#: ../DistUpgrade/DistUpgradeView.py:373
#, python-format
msgid ""
"\n"
"\n"
"You have to download a total of %s. "
msgstr ""
"\n"
"\n"
"Вам треба завантажити всього %s. "

#: ../DistUpgrade/DistUpgradeView.py:378
msgid ""
"Installing the upgrade can take several hours. Once the download has "
"finished, the process cannot be canceled."
msgstr ""
"Встановлення оновлень може зайняти декілька годин. Після завершення "
"завантаження процес неможливо скасувати."

#: ../DistUpgrade/DistUpgradeView.py:382
msgid ""
"Fetching and installing the upgrade can take several hours. Once the "
"download has finished, the process cannot be canceled."
msgstr ""
"Отримання та встановлення оновлень може зайняти декілька годин. Після "
"завершення завантаження процес неможливо скасувати."

#: ../DistUpgrade/DistUpgradeView.py:387
msgid "Removing the packages can take several hours. "
msgstr "Видалення пакунків може зайняти декілька годин. "

#. FIXME: this should go into DistUpgradeController
#: ../DistUpgrade/DistUpgradeView.py:392 ../UpdateManager/UpdateManager.py:676
msgid "The software on this computer is up to date."
msgstr "Програмне забезпечення на цьому комп'ютері актуальне."

#: ../DistUpgrade/DistUpgradeView.py:393
msgid ""
"There are no upgrades available for your system. The upgrade will now be "
"canceled."
msgstr "Немає оновлень для Вашої системи. Оновлення буде скасовано."

#: ../DistUpgrade/DistUpgradeView.py:406
msgid "Reboot required"
msgstr "Необхідно перезавантажити систему"

#: ../DistUpgrade/DistUpgradeView.py:407
msgid ""
"The upgrade is finished and a reboot is required. Do you want to do this now?"
msgstr ""
"Виконання оновлення завершено. Необхідно перезавантажити систему. "
"Перезавантажити зара?"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:72
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:72
#, python-format
msgid "authenticate '%(file)s' against '%(signature)s' "
msgstr "автентифікувати '%(file)s' замість '%(signature)s' "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:131
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:131
#, python-format
msgid "extracting '%s'"
msgstr "видобування '%s'"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:151
#: ../DistUpgrade/DistUpgradeFetcherCore.py:152
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:151
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:152
msgid "Could not run the upgrade tool"
msgstr "Неможливо провести апргрейд системи"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:152
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:152
msgid ""
"This is most likely a bug in the upgrade tool. Please report it as a bug "
"using the command 'ubuntu-bug update-manager'."
msgstr ""
"Швидше за все, це помилка в засобі оновлення. Повідомте, будь-ласка, про цю "
"помилку, використовуючи команду 'ubuntu-bug update-manager'."

#: ../DistUpgrade/DistUpgradeFetcherCore.py:227
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:227
msgid "Upgrade tool signature"
msgstr "Підпис інструменту оновлення"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:234
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:234
msgid "Upgrade tool"
msgstr "Інструмент оновлення"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:268
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:268
msgid "Failed to fetch"
msgstr "Не вдалося отримати"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:269
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:269
msgid "Fetching the upgrade failed. There may be a network problem. "
msgstr ""
"Не вдалося отримати оновлення. Це може бути викликано проблемою у мережі. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:273
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:273
msgid "Authentication failed"
msgstr "Помилка аутентифікації"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:274
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:274
msgid ""
"Authenticating the upgrade failed. There may be a problem with the network "
"or with the server. "
msgstr ""
"Автентифікація оновлення зазнала краху. Це може бути викликано проблемою з "
"мережею або сервером. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:279
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:279
msgid "Failed to extract"
msgstr "Не вдалося розпакувати"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:280
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:280
msgid ""
"Extracting the upgrade failed. There may be a problem with the network or "
"with the server. "
msgstr ""
"Не вдалося розпакувати оновлення. Можливо, виникла проблема в мережі або на "
"сервері. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:285
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:285
msgid "Verification failed"
msgstr "Перевірка не вдала"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:286
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:286
msgid ""
"Verifying the upgrade failed.  There may be a problem with the network or "
"with the server. "
msgstr ""
"Не вдалося перевірити наявність оновлень. Можливо, виникла проблема в мережі "
"або на сервері. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:300
#: ../DistUpgrade/DistUpgradeFetcherCore.py:306
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:300
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:306
msgid "Can not run the upgrade"
msgstr "Не вдається почати оновлення"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:301
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:301
msgid ""
"This usually is caused by a system where /tmp is mounted noexec. Please "
"remount without noexec and run the upgrade again."
msgstr ""
"Це, зазвичай, виникає в системах з /tmp змонтованому з noexec. Перемонтуйте, "
"будь-ласка, без noexec та виконайте оновлення знов."

#: ../DistUpgrade/DistUpgradeFetcherCore.py:307
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:307
#, python-format
msgid "The error message is '%s'."
msgstr "Повідомлення про помилку: '%s'."

#: ../DistUpgrade/DistUpgradeViewText.py:93
msgid ""
"Please report this as a bug and include the files /var/log/dist-upgrade/main."
"log and /var/log/dist-upgrade/apt.log in your report. The upgrade has "
"aborted.\n"
"Your original sources.list was saved in /etc/apt/sources.list.distUpgrade."
msgstr ""
"Будь ласка, повідомте про цю помилку та додайте файли  /var/log/dist-upgrade/"
"main.log та /var/log/dist-upgrade/apt.log до вашого звіту. Оновлення було "
"перервано.\n"
"Ваш оригінальний sources.list було збережено в /etc/apt/sources.list."
"distUpgrade."

#: ../DistUpgrade/DistUpgradeViewText.py:117
msgid "Aborting"
msgstr "Перервано"

#: ../DistUpgrade/DistUpgradeViewText.py:122
msgid "Demoted:\n"
msgstr "Понижено версію:\n"

#: ../DistUpgrade/DistUpgradeViewText.py:129
msgid "To continue please press [ENTER]"
msgstr "Щоб продовжити натисніть [ENTER]"

#: ../DistUpgrade/DistUpgradeViewText.py:157
#: ../DistUpgrade/DistUpgradeViewText.py:196
#: ../DistUpgrade/DistUpgradeViewText.py:203
msgid "Continue [yN] "
msgstr "Продовжити [yN] "

#: ../DistUpgrade/DistUpgradeViewText.py:157
#: ../DistUpgrade/DistUpgradeViewText.py:196
msgid "Details [d]"
msgstr "Подробиці [d]"

#. TRANSLATORS: the "y" is "yes"
#. TRANSLATORS: first letter of a positive (yes) answer
#: ../DistUpgrade/DistUpgradeViewText.py:162
#: ../DistUpgrade/DistUpgradeViewText.py:206
msgid "y"
msgstr "y"

#. TRANSLATORS: the "n" is "no"
#. TRANSLATORS: first letter of a negative (no) answer
#: ../DistUpgrade/DistUpgradeViewText.py:165
#: ../DistUpgrade/DistUpgradeViewText.py:213
msgid "n"
msgstr "n"

#. TRANSLATORS: the "d" is "details"
#: ../DistUpgrade/DistUpgradeViewText.py:168
msgid "d"
msgstr "d"

#: ../DistUpgrade/DistUpgradeViewText.py:173
#, python-format
msgid "No longer supported: %s\n"
msgstr "Більше не підтримується: %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:178
#, python-format
msgid "Remove: %s\n"
msgstr "Видалити: %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:188
#, python-format
msgid "Install: %s\n"
msgstr "Встановити %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:193
#, python-format
msgid "Upgrade: %s\n"
msgstr "Оновити %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:210
msgid "Continue [Yn] "
msgstr "Продовжити [Yn] "

#: ../DistUpgrade/DistUpgradeViewText.py:231
msgid ""
"To finish the upgrade, a restart is required.\n"
"If you select 'y' the system will be restarted."
msgstr ""
"Потрібне перезавантаження для завершення оновлення.\n"
"Оберіть 'y' для перезавантаження."

#: ../DistUpgrade/DistUpgrade.ui.h:1
msgid "_Cancel Upgrade"
msgstr "_Скасувати оновлення"

#: ../DistUpgrade/DistUpgrade.ui.h:2
msgid "_Resume Upgrade"
msgstr "Продовжити оновлення"

#: ../DistUpgrade/DistUpgrade.ui.h:3
msgid ""
"<b><big>Cancel the running upgrade?</big></b>\n"
"\n"
"The system could be in an unusable state if you cancel the upgrade. You are "
"strongly adviced to resume the upgrade."
msgstr ""
"<b><big>Відмінити оновлення системи?</big></b>\n"
"\n"
"Зауважте, що якщо Ви скасуєте оновлення, це може призвести до нестабільного "
"стану системи. Дуже рекомендується продовжити оновлення."

#: ../DistUpgrade/DistUpgrade.ui.h:6
msgid "_Start Upgrade"
msgstr "_Почати оновлення"

#: ../DistUpgrade/DistUpgrade.ui.h:9
msgid "_Replace"
msgstr "_Замінити"

#: ../DistUpgrade/DistUpgrade.ui.h:10
msgid "Difference between the files"
msgstr "Різниця між файлами"

#: ../DistUpgrade/DistUpgrade.ui.h:11
msgid "_Report Bug"
msgstr "Повідомити про помилку"

#: ../DistUpgrade/DistUpgrade.ui.h:12
msgid "_Continue"
msgstr "Пр_одовжити"

#: ../DistUpgrade/DistUpgrade.ui.h:13
msgid "<b><big>Start the upgrade?</big></b>"
msgstr "<b><big>Почати оновлення?</big></b>"

#: ../DistUpgrade/DistUpgrade.ui.h:15
msgid ""
"<b><big>Restart the system to complete the upgrade</big></b>\n"
"\n"
"Please save your work before continuing."
msgstr ""
"<b><big>Перезавантажте Вашу систему для завершення оновлення</big></b>\n"
"\n"
"Будь ласка, збережіть Вашу дані перед продовженням."

#: ../DistUpgrade/DistUpgrade.ui.h:18
msgid "Distribution Upgrade"
msgstr "Оновлення дистрибутивів"

#: ../DistUpgrade/DistUpgrade.ui.h:19
#, fuzzy
msgid "<b><big>Upgrading Ubuntu to version 12.10</big></b>"
msgstr "<b><big>Оновлення Ubuntu до версії 11.10</big></b>"

#: ../DistUpgrade/DistUpgrade.ui.h:20
msgid "    "
msgstr "    "

#: ../DistUpgrade/DistUpgrade.ui.h:22
msgid "Setting new software channels"
msgstr "Встановлення нових каналів програм"

#: ../DistUpgrade/DistUpgrade.ui.h:24
msgid "Restarting the computer"
msgstr "Перезавантаження системи"

#: ../DistUpgrade/DistUpgrade.ui.h:27
msgid "Terminal"
msgstr "Термінал"

#: ../UpdateManager/backend/InstallBackendSynaptic.py:64
msgid "Please wait, this can take some time."
msgstr "Будь ласка, зачекайте, це може зайняти деякий час."

#: ../UpdateManager/backend/InstallBackendSynaptic.py:66
msgid "Update is complete"
msgstr "Оновлення завершено"

#: ../UpdateManager/DistUpgradeFetcher.py:114
#: ../UpdateManager/DistUpgradeFetcherKDE.py:109
msgid "Could not find the release notes"
msgstr "Не вдалося знайти примітки випуску."

#: ../UpdateManager/DistUpgradeFetcher.py:115
#: ../UpdateManager/DistUpgradeFetcherKDE.py:110
msgid "The server may be overloaded. "
msgstr "Сервер може бути перенавантажений. "

#: ../UpdateManager/DistUpgradeFetcher.py:125
#: ../UpdateManager/DistUpgradeFetcherKDE.py:114
msgid "Could not download the release notes"
msgstr "Не вдалося завантажити примітки випуску."

#: ../UpdateManager/DistUpgradeFetcher.py:126
#: ../UpdateManager/DistUpgradeFetcherKDE.py:115
msgid "Please check your internet connection."
msgstr "Будь ласка, перевірте ваше з'єднання з Інтернетом."

#: ../UpdateManager/DistUpgradeFetcherKDE.py:68
#: ../UpdateManager/DistUpgradeFetcherKDE.py:91
msgid "Upgrade"
msgstr "Оновлення"

#: ../UpdateManager/DistUpgradeFetcherKDE.py:95
#: ../data/gtkbuilder/UpdateManager.ui.h:20
#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:2
msgid "Release Notes"
msgstr "Відомості про релізе"

#: ../UpdateManager/DistUpgradeFetcherKDE.py:134
#: ../UpdateManager/DistUpgradeFetcherKDE.py:148
#: ../UpdateManager/DistUpgradeFetcherKDE.py:150
msgid "Downloading additional package files..."
msgstr "Завантаження додаткових файлів пакунків..."

#: ../UpdateManager/DistUpgradeFetcherKDE.py:148
#, python-format
msgid "File %s of %s at %sB/s"
msgstr "Файл %s з %s при %sБ/с"

#: ../UpdateManager/DistUpgradeFetcherKDE.py:150
#, python-format
msgid "File %s of %s"
msgstr "Файл %s з %s"

#: ../UpdateManager/ChangelogViewer.py:75
msgid "Open Link in Browser"
msgstr "Відкрити посилання в браузері"

#: ../UpdateManager/ChangelogViewer.py:78
msgid "Copy Link to Clipboard"
msgstr "Копіювати посилання в буфер обміну"

#: ../UpdateManager/GtkProgress.py:162
#, python-format
msgid "Downloading file %(current)li of %(total)li with %(speed)s/s"
msgstr "Завантаження файлу %(current)li з %(total)li із швидкістю %(speed)s/с"

#: ../UpdateManager/GtkProgress.py:167
#, python-format
msgid "Downloading file %(current)li of %(total)li"
msgstr "Завантаження файлу %(current)li з %(total)li"

#: ../UpdateManager/UpdateManager.py:106 ../do-release-upgrade:100
msgid "Your Ubuntu release is not supported anymore."
msgstr "Ваша версія Ubuntu вже не підтримується."

#: ../UpdateManager/UpdateManager.py:107
msgid ""
"You will not get any further security fixes or critical updates. Please "
"upgrade to a later version of Ubuntu."
msgstr ""
"Ви надалі не отримуватимете жодних виправлень безпеки та критичних оновлень. "
"Будь ласка, оновіть систему до пізнішої версії Ubuntu."

#: ../UpdateManager/UpdateManager.py:115
msgid "Upgrade information"
msgstr "Інформація про оновлення"

#: ../UpdateManager/UpdateManager.py:233
#: ../UpdateManagerText/UpdateManagerText.py:35
msgid "Install"
msgstr "Встановлення"

#: ../UpdateManager/UpdateManager.py:235
msgid "Name"
msgstr "Назва"

#. upload_archive = version_match.group(2).strip()
#: ../UpdateManager/UpdateManager.py:395
#, python-format
msgid "Version %s: \n"
msgstr "Версія %s: \n"

#: ../UpdateManager/UpdateManager.py:453
msgid ""
"No network connection detected, you can not download changelog information."
msgstr ""
"Ви не можете завантажити список змін за відсутності з’єднання з інтернетом."

#: ../UpdateManager/UpdateManager.py:463
msgid "Downloading list of changes..."
msgstr "Завантаження списку змін..."

#: ../UpdateManager/UpdateManager.py:507
msgid "_Deselect All"
msgstr "_Прибрати виділення"

#: ../UpdateManager/UpdateManager.py:513
msgid "Select _All"
msgstr "Вибрати _все"

#: ../UpdateManager/UpdateManager.py:572
#, python-format
msgid "%s will be downloaded."
msgstr "%s буде завантажено."

#: ../UpdateManager/UpdateManager.py:584
#, fuzzy
msgid "The update has already been downloaded."
msgid_plural "The updates have already been downloaded."
msgstr[0] "Оновлення завантажено, але не встановлено."
msgstr[1] "Оновлення завантажено, але не встановлено."
msgstr[2] "Оновлення завантажено, але не встановлено."

#: ../UpdateManager/UpdateManager.py:589
msgid "There are no updates to install."
msgstr "Немає оновлень для встановлення."

#: ../UpdateManager/UpdateManager.py:598
msgid "Unknown download size."
msgstr "Завантажувальний розмір невідомий."

#: ../UpdateManager/UpdateManager.py:624
msgid ""
"It is unknown when the package information was updated last. Please click "
"the 'Check' button to update the information."
msgstr ""
"Неможливо з’ясувати коли було востаннє оновлено інформацію про пакунки. "
"Натисніть, будь-ласка, кнопку \"Перевірити\" щоб зробити це зараз."

#: ../UpdateManager/UpdateManager.py:630
#, python-format
msgid ""
"The package information was last updated %(days_ago)s days ago.\n"
"Press the 'Check' button below to check for new software updates."
msgstr ""
"Інформація про пакунки востаннє була оновлена %(days_ago)s днів тому.\n"
"Натисніть кнопку \"Перевірка\" для перевірки наявності оновлень."

#: ../UpdateManager/UpdateManager.py:635
#, python-format
msgid "The package information was last updated %(days_ago)s day ago."
msgid_plural "The package information was last updated %(days_ago)s days ago."
msgstr[0] "Інформація про пакунки була оновлена %(days_ago)s день тому."
msgstr[1] "Інформація про пакунки була оновлена %(days_ago)s днів тому."
msgstr[2] "Інформація про пакунки була оновлена %(days_ago)s днів тому."

#: ../UpdateManager/UpdateManager.py:639
#, python-format
msgid "The package information was last updated %(hours_ago)s hour ago."
msgid_plural ""
"The package information was last updated %(hours_ago)s hours ago."
msgstr[0] "Інформація про пакунки була оновлена %(hours_ago)s годину тому."
msgstr[1] "Інформація про пакунки була оновлена %(hours_ago)s годин тому."
msgstr[2] "Інформація про пакунки була оновлена %(hours_ago)s годин тому."

#. TRANSLATORS: only in plural form, as %s minutes ago is one of 15, 30, 45 minutes ago
#: ../UpdateManager/UpdateManager.py:644 ../UpdateManager/UpdateManager.py:646
#: ../UpdateManager/UpdateManager.py:648
#, python-format
msgid "The package information was last updated about %s minutes ago."
msgstr "Інформація про пакунки востаннє була оновлена близько %s хвилин тому."

#: ../UpdateManager/UpdateManager.py:650
msgid "The package information was just updated."
msgstr "Інформація про пакунки була щойно оновлена."

#: ../UpdateManager/UpdateManager.py:689
msgid "Software updates may be available for your computer."
msgstr ""
"Для вашого комп'ютера можуть бути доступні оновлення програмного "
"забезпечення."

#: ../UpdateManager/UpdateManager.py:697
#, python-format
msgid ""
"Updated software has been issued since %s was released. Do you want to "
"install it now?"
msgstr ""

#: ../UpdateManager/UpdateManager.py:700
msgid ""
"Updated software is available for this computer. Do you want to install it "
"now?"
msgstr ""

#: ../UpdateManager/UpdateManager.py:758
#, python-format
msgid ""
"The upgrade needs a total of %s free space on disk '%s'. Please free at "
"least an additional %s of disk space on '%s'. Empty your trash and remove "
"temporary packages of former installations using 'sudo apt-get clean'."
msgstr ""
"Оновлення потребує всього %s вільного місця на диску '%s'. Будь-ласка, "
"звільніть принаймні %s додатково дискового простору на '%s'. Очистіть "
"смітник і видаліть тимчасові пакети колишніх об'єктів за допомогою 'sudo apt-"
"get clean'."

#: ../UpdateManager/UpdateManager.py:783
msgid ""
"The computer needs to restart to finish installing updates. Please save your "
"work before continuing."
msgstr ""
"Комп’ютер потребує перезавантаження для завершення встановлення оновлень. "
"Збережіть ваші дані перед тим як продовжити."

#: ../UpdateManager/UpdateManager.py:847
msgid "Reading package information"
msgstr "Зчитування інформації про пакунок"

#: ../UpdateManager/UpdateManager.py:862
msgid "Connecting..."
msgstr "З’єднання..."

#: ../UpdateManager/UpdateManager.py:879
msgid "You may not be able to check for updates or download new updates."
msgstr "Ви не можете перевірити наявність оновлень або завантажити оновлення."

#: ../UpdateManager/UpdateManager.py:1002
msgid "Could not initialize the package information"
msgstr "Неможливо ініціалізувати інформацію про пакунок"

#: ../UpdateManager/UpdateManager.py:1003
msgid ""
"An unresolvable problem occurred while initializing the package "
"information.\n"
"\n"
"Please report this bug against the 'update-manager' package and include the "
"following error message:\n"
msgstr ""
"В процесі ініціалізації інформації про пакунок виникла проблема, яку "
"неможливо розв’язати.\n"
"\n"
"Будь ласка, повідомте про цю помилку в \"Менеджері оновлення\" та включіть у "
"звіт таке повідомлення:\n"

#: ../UpdateManager/UpdateManager.py:1032
msgid ""
"An unresolvable problem occurred while calculating the upgrade.\n"
"\n"
"Please report this bug against the 'update-manager' package and include the "
"following error message:"
msgstr ""
"В процесі обробки інформації про оновлення виникла проблема, яку неможливо "
"розв’язати.\n"
"\n"
"Будь ласка, повідомте про цю помилку в \"Менеджері оновлення\" та включіть у "
"звіт таке повідомлення:"

#: ../UpdateManager/UpdateManager.py:1056
msgid " (New install)"
msgstr " (Нове встановлення)"

#. TRANSLATORS: the b stands for Bytes
#: ../UpdateManager/UpdateManager.py:1063
#, python-format
msgid "(Size: %s)"
msgstr "(Розмір %s)"

#: ../UpdateManager/UpdateManager.py:1067
#, python-format
msgid "From version %(old_version)s to %(new_version)s"
msgstr "Від версії %(old_version)s до %(new_version)s"

#: ../UpdateManager/UpdateManager.py:1071
#, python-format
msgid "Version %s"
msgstr "Версія %s"

#: ../UpdateManager/UpdateManager.py:1104 ../do-release-upgrade:112
msgid "Release upgrade not possible right now"
msgstr "Оновлення реліза зараз неможливо."

#: ../UpdateManager/UpdateManager.py:1105 ../do-release-upgrade:113
#, c-format, python-format
msgid ""
"The release upgrade can not be performed currently, please try again later. "
"The server reported: '%s'"
msgstr ""
"Оновлення реліза не може бути виконано зараз, будь ласка, спробуйте пізніше. "
"Сервер повідомив: '%s'"

#: ../UpdateManager/UpdateManager.py:1107 ../check-new-release-gtk:126
msgid "Downloading the release upgrade tool"
msgstr "Завантаження засобу оновлення випуску"

#: ../UpdateManager/UpdateManager.py:1114
#, python-format
msgid "<b>New Ubuntu release '%s' is available</b>"
msgstr "<b>Новий випуск Ubuntu '%s' є доступним</b>"

#. we assert a clean cache
#: ../UpdateManager/UpdateManager.py:1153
msgid "Software index is broken"
msgstr "Індекс програм пошкоджений"

#: ../UpdateManager/UpdateManager.py:1154
msgid ""
"It is impossible to install or remove any software. Please use the package "
"manager \"Synaptic\" or run \"sudo apt-get install -f\" in a terminal to fix "
"this issue at first."
msgstr ""
"Встановлення або видалення програм неможливе. Для виправлення цієї ситуації "
"використовуйте менеджер пакетів \"Synaptic\" або запустіть в терміналі "
"\"sudo apt-get install -f\"."

#: ../UpdateManager/UnitySupport.py:57
msgid "Check for Updates"
msgstr "Перевірити наявність оновлень"

#: ../UpdateManager/UnitySupport.py:66
msgid "Install All Available Updates"
msgstr "Встановити всі доступні оновлення"

#: ../UpdateManagerText/UpdateManagerText.py:34
msgid "Cancel"
msgstr "Скасувати"

#: ../UpdateManagerText/UpdateManagerText.py:37
msgid "Changelog"
msgstr "Журнал змін"

#: ../UpdateManagerText/UpdateManagerText.py:40
msgid "Updates"
msgstr "Оновлення"

#: ../UpdateManagerText/UpdateManagerText.py:53
msgid "Building Updates List"
msgstr "Формується список оновлень"

#: ../UpdateManagerText/UpdateManagerText.py:56
msgid ""
"\n"
"A normal upgrade can not be calculated, please run: \n"
"  sudo apt-get dist-upgrade\n"
"\n"
"\n"
"This can be caused by:\n"
" * A previous upgrade which didn't complete\n"
" * Problems with some of the installed software\n"
" * Unofficial software packages not provided by Ubuntu\n"
" * Normal changes of a pre-release version of Ubuntu"
msgstr ""
"\n"
"Звичайне оновлення не може бути оброблене, будь-ласка запустіть: \n"
"  sudo apt-get dist-upgrade\n"
"\n"
"\n"
"Це може бути спричинене:\n"
" * Незавершеним останнім оновленням\n"
" * Проблемами з деяким встановленим програмним забезпеченням\n"
" * Неофіційним програмним забезпеченням, яке не підтримується Ubuntu\n"
" * Звичайними змінами у попередніх випусках Ubuntu"

#: ../UpdateManagerText/UpdateManagerText.py:125
msgid "Downloading changelog"
msgstr "Завантаження списку змін"

#: ../UpdateManager/Core/MyCache.py:147
#, python-format
msgid "Other updates (%s)"
msgstr "Інші оновлення (%s)"

#: ../UpdateManager/Core/MyCache.py:325
msgid "This update does not come from a source that supports changelogs."
msgstr "Це оновлення надходить з джерела, що не підтримує журналювання змін."

#: ../UpdateManager/Core/MyCache.py:331 ../UpdateManager/Core/MyCache.py:359
msgid ""
"Failed to download the list of changes. \n"
"Please check your Internet connection."
msgstr ""
"Помилка завантаження списку змін. \n"
"Перевірте з'єднання з Internet."

#: ../UpdateManager/Core/MyCache.py:338
#, python-format
msgid ""
"Changes for the versions:\n"
"Installed version: %s\n"
"Available version: %s\n"
"\n"
msgstr ""
"Зміни версій:\n"
"Встановлена версія: %s\n"
"Доступна версія: %s\n"
"\n"

#: ../UpdateManager/Core/MyCache.py:348
#, python-format
msgid ""
"The changelog does not contain any relevant changes.\n"
"\n"
"Please use http://launchpad.net/ubuntu/+source/%s/%s/+changelog\n"
"until the changes become available or try again later."
msgstr ""
"Журнал змін не містить істотних змін.\n"
"\n"
"Будь ласка, використовуйте http://launchpad.net/ubuntu/+source/%s/%s/"
"+changelog,\n"
"доки зміни не стануть доступними, або спробуйте знову пізніше."

#: ../UpdateManager/Core/MyCache.py:353
#, python-format
msgid ""
"The list of changes is not available yet.\n"
"\n"
"Please use http://launchpad.net/ubuntu/+source/%s/%s/+changelog\n"
"until the changes become available or try again later."
msgstr ""
"Журнал змін ще не доступний.\n"
"\n"
"Будь ласка, використовуйте http://launchpad.net/ubuntu/+source/%s/%s/"
"+changelog,\n"
"доки зміни не стануть доступними, або спробуйте знову пізніше."

#: ../UpdateManager/Core/UpdateList.py:51
msgid "Failed to detect distribution"
msgstr "Не вдалося визначити дистрибутив"

#: ../UpdateManager/Core/UpdateList.py:52
#, python-format
msgid "A error '%s' occurred while checking what system you are using."
msgstr "Виникла помилка '%s' при перевірці яку систему Ви використовуєте."

#: ../UpdateManager/Core/UpdateList.py:63
msgid "Important security updates"
msgstr "Важливі оновлення безпеки"

#: ../UpdateManager/Core/UpdateList.py:64
msgid "Recommended updates"
msgstr "Рекоменодовані оновлення"

#: ../UpdateManager/Core/UpdateList.py:65
msgid "Proposed updates"
msgstr "Пропоновані оновлення"

#: ../UpdateManager/Core/UpdateList.py:66
msgid "Backports"
msgstr "Backports"

#: ../UpdateManager/Core/UpdateList.py:67
msgid "Distribution updates"
msgstr "Оновлення дистрибутиву"

#: ../UpdateManager/Core/UpdateList.py:72
msgid "Other updates"
msgstr "Інші оновлення"

#: ../data/gtkbuilder/UpdateManager.ui.h:1
#, fuzzy
msgid "<big><b>Starting Software Updater</b></big>"
msgstr "<b><big>Запуск менеджера оновлень</big></b>"

#: ../data/gtkbuilder/UpdateManager.ui.h:2
msgid ""
"Software updates correct errors, eliminate security vulnerabilities and "
"provide new features."
msgstr ""
"Оновлення програм виправляють помилки, проблеми безпеки та додають нові "
"можливості."

#: ../data/gtkbuilder/UpdateManager.ui.h:3
msgid "_Partial Upgrade"
msgstr "_Часткове оновлення"

#: ../data/gtkbuilder/UpdateManager.ui.h:4
msgid "<big><b>Not all updates can be installed</b></big>"
msgstr "<big><b>Не всі оновлення можливо встановити</b></big>"

#: ../data/gtkbuilder/UpdateManager.ui.h:5
msgid ""
"Run a partial upgrade, to install as many updates as possible. \n"
"\n"
"This can be caused by:\n"
" * A previous upgrade which didn't complete\n"
" * Problems with some of the installed software\n"
" * Unofficial software packages not provided by Ubuntu\n"
" * Normal changes of a pre-release version of Ubuntu"
msgstr ""
"Виконати часткове оновлення, що здійснить максимально можливе встановлення "
"оновлень. \n"
"\n"
"Це може бути викликано:\n"
" * попереднім оновленням, яке ще не завершено\n"
" * проблемами з деяким встановленими програмами\n"
" * неофіційними пакунками програм, що не постачаються з Ubuntu\n"
" * нормальними змінами пре-реліз версії Ubuntu"

#: ../data/gtkbuilder/UpdateManager.ui.h:12
msgid "Chec_k"
msgstr "Перевір_ка"

#: ../data/gtkbuilder/UpdateManager.ui.h:13
msgid ""
"<b><big>You must check for updates manually</big></b>\n"
"\n"
"Your system does not check for updates automatically. You can configure this "
"behavior in <i>Software Sources</i> on the <i>Updates</i> tab."
msgstr ""
"<b><big>Вам потрібно перевірити наявність оновлень вручну</big></b>\n"
"\n"
"Ваша система не перевіряє наявність оновлень автоматично. Ви можете змінити "
"цю поведінку у <i>Джерелах програмного забезпечення</i> на вкладці "
"<i>Оновлення</i>."

#: ../data/gtkbuilder/UpdateManager.ui.h:16
msgid "_Hide this information in the future"
msgstr "_Не показувати цю інформацію надалі"

#: ../data/gtkbuilder/UpdateManager.ui.h:17
msgid "Co_ntinue"
msgstr "Про_довжити"

#: ../data/gtkbuilder/UpdateManager.ui.h:18
msgid "<big><b>Running on battery</b></big>"
msgstr "<big><b>Працює від акумулятора</b></big>"

#: ../data/gtkbuilder/UpdateManager.ui.h:19
msgid "Your system is running on battery. Are you sure you want to continue?"
msgstr ""
"Ваша система працює від акумулятора. Ви впевнені, що бажаєте продовжити?"

#: ../data/gtkbuilder/UpdateManager.ui.h:21
msgid "_Upgrade"
msgstr "_Оновлення"

#: ../data/gtkbuilder/UpdateManager.ui.h:22
#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:8
msgid "Show progress of individual files"
msgstr "Показувати прогрес для окремих файлів"

#: ../data/gtkbuilder/UpdateManager.ui.h:23
#: ../data/update-manager.desktop.in.h:1
#, fuzzy
msgid "Software Updater"
msgstr "Оновлення програм"

#: ../data/gtkbuilder/UpdateManager.ui.h:24
#, fuzzy
msgid "Starting Software Updater"
msgstr "Оновлення програм"

#: ../data/gtkbuilder/UpdateManager.ui.h:25
msgid "U_pgrade"
msgstr "Оновити"

#: ../data/gtkbuilder/UpdateManager.ui.h:27
msgid "updates"
msgstr "оновлення"

#: ../data/gtkbuilder/UpdateManager.ui.h:28
msgid "Changes"
msgstr "Зміни"

#: ../data/gtkbuilder/UpdateManager.ui.h:29
msgid "Description"
msgstr "Опис"

#: ../data/gtkbuilder/UpdateManager.ui.h:30
#, fuzzy
msgid "Details of updates"
msgstr "Опис оновлень"

#: ../data/gtkbuilder/UpdateManager.ui.h:31
msgid ""
"You are connected via roaming and may be charged for the data consumed by "
"this update."
msgstr ""
"Ви під'єднані в режимі роумінгу, тому може стягуватися додаткова плата за "
"дані, передані під час цього оновлення."

#: ../data/gtkbuilder/UpdateManager.ui.h:32
msgid ""
"You may want to wait until you’re not using a mobile broadband connection."
msgstr ""

#: ../data/gtkbuilder/UpdateManager.ui.h:33
msgid "It’s safer to connect the computer to AC power before updating."
msgstr ""
"Безпечніше буде під'єднати комп'ютер до мережі змінного струму перед "
"оновленням."

#: ../data/gtkbuilder/UpdateManager.ui.h:34
msgid "_Settings..."
msgstr "_Параметри…"

#: ../data/gtkbuilder/UpdateManager.ui.h:35
#, fuzzy
msgid "_Install Now"
msgstr "Встановлення"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:1
msgid "<b>A new version of Ubuntu is available. Would you like to upgrade?</b>"
msgstr "<b>Доступна нова версія Ubuntu. Хочете оновити?</b>"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:3
msgid "Don't Upgrade"
msgstr "Не оновлювати"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:4
msgid "Ask Me Later"
msgstr "Запитати пізніше"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:5
msgid "Yes, Upgrade Now"
msgstr "Так, оновити зараз"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:6
msgid "You have declined to upgrade to the new Ubuntu"
msgstr "Ви відмовилось від оновлення Ubuntu до нової версії"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:7
#, fuzzy
msgid ""
"You can upgrade at a later time by opening Software Updater and click on "
"\"Upgrade\"."
msgstr ""
"Ви можете оновити систему пізніше, відкривши Менеджер Оновлень і натиснув "
"«Встановити оновлення»"

#: ../data/update-manager.desktop.in.h:2
msgid "Software Updates"
msgstr "Оновлення програм"

#: ../data/update-manager.desktop.in.h:3
msgid "Show and install available updates"
msgstr "Перегляд і встановлення доступних оновлень"

#: ../update-manager:66 ../update-manager-text:55 ../do-release-upgrade:51
msgid "Show version and exit"
msgstr "Показати версію та вийти"

#: ../update-manager:69
msgid "Directory that contains the data files"
msgstr "Каталог, який вміщує файли даних."

#: ../update-manager:72
msgid "Check if a new Ubuntu release is available"
msgstr "Перевірити чи доступний новий випуск Ubuntu"

#: ../update-manager:75 ../do-release-upgrade:54 ../check-new-release-gtk:186
msgid "Check if upgrading to the latest devel release is possible"
msgstr ""
"Перевірити можливість оновлення до останнього випуску, що перебуває в стані "
"розробки"

#: ../update-manager:79
msgid "Upgrade using the latest proposed version of the release upgrader"
msgstr ""
"Оновити, використовуючи найновішу, запропоновану оновлювачем випусків, версію"

#: ../update-manager:86
msgid "Do not focus on map when starting"
msgstr "Не вибирати карту при запуску"

#: ../update-manager:89
msgid "Try to run a dist-upgrade"
msgstr "Спробуйте запустить dist-upgrade"

#: ../update-manager:92
msgid "Do not check for updates when starting"
msgstr "Не перевіряти оновлення під час старту"

#: ../update-manager:96 ../do-release-upgrade:70
msgid "Test upgrade with a sandbox aufs overlay"
msgstr "Протестувати оновлення в безпечному режимі"

#: ../update-manager:116
msgid "Running partial upgrade"
msgstr "Проводиться часткове оновлення"

#: ../update-manager-text:59
msgid "Show description of the package instead of the changelog"
msgstr "Показати опис пакета замість журналу змін."

#: ../do-release-upgrade:58 ../check-new-release-gtk:190
msgid ""
"Try upgrading to the latest release using the upgrader from $distro-proposed"
msgstr "Спробуйте оновитись до останньої версії за допомогою $distro-proposed"

#: ../do-release-upgrade:62
msgid ""
"Run in a special upgrade mode.\n"
"Currently 'desktop' for regular upgrades of a desktop system and 'server' "
"for server systems are supported."
msgstr ""
"Запустити в особливому режимі оновлення.\n"
"Наразі підтримуються режим \"стільниця\" для регулярного оновлення "
"персональних робочих станцій та \"сервер\" для серверних систем."

#: ../do-release-upgrade:68
msgid "Run the specified frontend"
msgstr "Запустити вказаний інтерфейс оболонки"

#: ../do-release-upgrade:73
msgid ""
"Check only if a new distribution release is available and report the result "
"via the exit code"
msgstr ""
"Перевіряти доступність нової версії дистрибутива і повертати результат за "
"допомогою кода виходу."

#: ../do-release-upgrade:87
msgid "Checking for a new Ubuntu release"
msgstr "Перевіряється наявність нового випуску Ubuntu"

#: ../do-release-upgrade:101
msgid ""
"For upgrade information, please visit:\n"
"%(url)s\n"
msgstr ""
"Для отримання інформації щодо оновлення, будь ласка, відвідайте:\n"
"%(url)s\n"

#: ../do-release-upgrade:107
msgid "No new release found"
msgstr "Нової версії не знайдено"

#: ../do-release-upgrade:119
#, c-format
msgid "New release '%s' available."
msgstr "Доступний новий реліз '%s' ."

#: ../do-release-upgrade:120
msgid "Run 'do-release-upgrade' to upgrade to it."
msgstr "Виконайте команду 'do-release-upgrade' для оновлення."

#: ../check-new-release-gtk:101
msgid "Ubuntu %(version)s Upgrade Available"
msgstr "Доступне оновлення Ubuntu %(version)s"

#: ../check-new-release-gtk:143
#, c-format
msgid "You have declined the upgrade to Ubuntu %s"
msgstr "Ви відмовились від оновлення до Ubuntu %s"

#: ../check-new-release-gtk:196
msgid "Add debug output"
msgstr "Додати результати відладки"

#: ../ubuntu-support-status:91
msgid "Show unsupported packages on this machine"
msgstr "Показати непідтримувані пакунки на цьому комп'ютері."

#: ../ubuntu-support-status:94
msgid "Show supported packages on this machine"
msgstr "Показати підтримувані пакунки на цьому комп'ютері."

#: ../ubuntu-support-status:97
msgid "Show all packages with their status"
msgstr "Показати всі пакунки з їх статусами"

#: ../ubuntu-support-status:100
msgid "Show all packages in a list"
msgstr "Показати список всіх пакунків"

#: ../ubuntu-support-status:142
#, c-format
msgid "Support status summary of '%s':"
msgstr "Сумарний статус підтримки '%s':"

#: ../ubuntu-support-status:145
msgid "You have %(num)s packages (%(percent).1f%%) supported until %(time)s"
msgstr ""
"Ви маєте %(num)s пакунків  (%(percent).1f%%) з них підтримуються до %(time)s"

#: ../ubuntu-support-status:151
msgid ""
"You have %(num)s packages (%(percent).1f%%) that can not/no-longer be "
"downloaded"
msgstr ""
"Ви маєте %(num)s пакунків (%(percent).1f%%) яких вже/взагалі не можуть бути "
"завантажені."

#: ../ubuntu-support-status:154
msgid "You have %(num)s packages (%(percent).1f%%) that are unsupported"
msgstr "Ви маєте %(num)s пакунків (%(percent).1f%%) яких не підтримуються."

#: ../ubuntu-support-status:162
msgid ""
"Run with --show-unsupported, --show-supported or --show-all to see more "
"details"
msgstr ""
"Запустить з --show-unsupported, --show-supported або --show-all, щоб "
"подивитьсь більш детально"

#: ../ubuntu-support-status:166
msgid "No longer downloadable:"
msgstr "Більше не доступне для завантаження"

#: ../ubuntu-support-status:169
msgid "Unsupported: "
msgstr "Не підтримується: "

#: ../ubuntu-support-status:174
#, c-format
msgid "Supported until %s:"
msgstr "Підтримується до %s:"

#: ../ubuntu-support-status:183
msgid "Unsupported"
msgstr "Не підтримується"

#. Why do we use %s here instead of $strings or {} format placeholders?
#. It's because we don't want to break existing translations.
#: ../janitor/plugincore/exceptions.py:42
#, python-format
msgid "Unimplemented method: %s"
msgstr "Незадіяний метод: %s"

#: ../janitor/plugincore/core/file_cruft.py:41
msgid "A file on disk"
msgstr "Файл на диску"

#: ../janitor/plugincore/core/missing_package_cruft.py:39
msgid "Install missing package."
msgstr "Встановлення відсутніх пакунків."

#. 2012-06-08 BAW: i18n string; don't use {} or PEP 292.
#: ../janitor/plugincore/core/missing_package_cruft.py:49
#, python-format
msgid "Package %s should be installed."
msgstr "Пакунок %s повинен бути встановлений."

#: ../janitor/plugincore/core/package_cruft.py:49
msgid ".deb package"
msgstr ".deb пакунок"

#: ../janitor/plugincore/plugins/langpack_manual_plugin.py:46
#, python-format
msgid "%s needs to be marked as manually installed."
msgstr "%s повинен бути позначений як встановлений вручну ."

#: ../janitor/plugincore/plugins/kdelibs4to5_plugin.py:49
msgid ""
"When upgrading, if kdelibs4-dev is installed, kdelibs5-dev needs to be "
"installed. See bugs.launchpad.net, bug #279621 for details."
msgstr ""
"Під час оновлення, якщо встановлено kdelibs4-dev, також повинно бути "
"встановлено kdelibs5-dev. Див. bugs.launchpad.net, bug #279621 для детальної "
"інформації."

#: ../janitor/plugincore/plugins/dpkg_status_plugin.py:44
#, python-format
msgid "%i obsolete entries in the status file"
msgstr "%i застарілі записи в файлі статусу"

#: ../janitor/plugincore/plugins/dpkg_status_plugin.py:47
msgid "Obsolete entries in dpkg status"
msgstr "Застарілі записи в dpkg статусі"

#. pragma: no cover
#: ../janitor/plugincore/plugins/dpkg_status_plugin.py:50
msgid "Obsolete dpkg status entries"
msgstr "Застарілі записи dpkg статусів"

#: ../janitor/plugincore/plugins/remove_lilo_plugin.py:42
msgid "Remove lilo since grub is also installed.(See bug #314004 for details.)"
msgstr ""
"Видаліть LILO після того, як встановлено GRUB. (Дивіться помилку #314004 для "
"більш детальної інформації)."

#~ msgid ""
#~ "After your package information was updated the essential package '%s' can "
#~ "not be found anymore so a bug reporting process is being started."
#~ msgstr ""
#~ "Після того, як пакет '%s' був оновлений, його не вдається знайти, тому "
#~ "процес звітування про помилку розпочався."

#~ msgid "<b><big>Upgrading Ubuntu to version 12.04</big></b>"
#~ msgstr "<b><big>Оновлення Ubuntu до версії 12.04</big></b>"

#~ msgid "%(count)s update has been selected."
#~ msgid_plural "%(count)s updates have been selected."
#~ msgstr[0] "Обрано %(count)s оновлення."
#~ msgstr[1] "Обрано %(count)s оновлення."
#~ msgstr[2] "Обрано %(count)s оновлень."

#~ msgid "%(count_str)s %(download_str)s"
#~ msgstr "%(count_str)s %(download_str)s"

#~ msgid "Welcome to Ubuntu"
#~ msgstr "Ласкаво просимо в Ubuntu"

#~ msgid ""
#~ "These software updates have been issued since this version of Ubuntu was "
#~ "released."
#~ msgstr "Ці оновлення було випущено з моменту випуску цієї версії Ubuntu."

#~ msgid "Software updates are available for this computer."
#~ msgstr "Для цього комп’ютера доступні оновлення програмного забезпечення."

#~ msgid "Update Manager"
#~ msgstr "Менеджер оновлення"

#~ msgid "Starting Update Manager"
#~ msgstr "Запуск менеджера оновлень."

#~ msgid "You are connected via a wireless modem."
#~ msgstr "Ви під’єднані за допомогою бездротового модему."

#~ msgid "_Install Updates"
#~ msgstr "_Встановлення оновлень"

#~ msgid "Checking for a new ubuntu release"
#~ msgstr "Перевірка нової версії Ubuntu"

#~ msgid "Your system is up-to-date"
#~ msgstr "Ваша система оновлена!"

#~ msgid "Software updates are available for this computer"
#~ msgstr "Оновлення програмного забезпечення доступні на цьому комп'ютері"

#~ msgid ""
#~ "If you don't want to install them now, choose \"Update Manager\" from the "
#~ "Administration menu later."
#~ msgstr ""
#~ "Якщо ви не хочете встановити їх зараз, виберіть \"Менеджер оновлення\" з "
#~ "меню Адміністрування пізніше."

#~ msgid "There are no updates to install"
#~ msgstr "Оновлення для установки відсутні."

#~ msgid "The update has already been downloaded, but not installed"
#~ msgid_plural "The updates have already been downloaded, but not installed"
#~ msgstr[0] "Оновлення вже завантажено, але ще не встановлено."
#~ msgstr[1] "Оновлення вже завантажені, але ще не встановлені."
#~ msgstr[2] "Оновлення вже завантажені, але ще не встановлені."

#~ msgid ""
#~ "You will not get any further security fixes or critical updates. Please "
#~ "upgrade to a later version of Ubuntu Linux."
#~ msgstr ""
#~ "Ви більше не зможете отримувати оновлень безпеки та критичних оновлень. "
#~ "Будь ласка, виконайте оновлення до останньої версії Ubuntu Linux."

#~ msgid ""
#~ "This upgrade is running in sandbox (test) mode. All changes are written "
#~ "to '%s' and will be lost on the next reboot.\n"
#~ "\n"
#~ "*No* changes written to a systemdir from now until the next reboot are "
#~ "permanent."
#~ msgstr ""
#~ "Це оновлення запущено в тестовому режимі (пісочниці). Усі зміни записані "
#~ "в '%s' і будуть втрачені після перезавантаження.\n"
#~ "\n"
#~ "*Жодні* змінити, записані в системний каталог відтепер і до наступного "
#~ "перезавантаження не будуть постійними ."

#~ msgid ""
#~ "Fetching and installing the upgrade can take several hours. Once the "
#~ "download has finished, the process cannot be cancelled."
#~ msgstr ""
#~ "Отримання та встановлення оновлень може тривати кілька годин. Коли процес "
#~ "завершиться, його вже не можна буде скасувати."

#~ msgid ""
#~ "If you don't want to install them now, choose \"Update Manager\" from "
#~ "Applications later."
#~ msgstr ""
#~ "Якщо ви не хочете встановлювати їх зараз, скористайтеся \"Менеджером "
#~ "оновлення \" з меню Програми пізніше."

#~ msgid ""
#~ "After your package information was updated the essential package '%s' can "
#~ "not be found anymore.\n"
#~ "This indicates a serious error, please report this bug using the command "
#~ "'ubuntu-bug update-manager' in a terminal and include the files in /var/"
#~ "log/dist-upgrade/ in the bug report."
#~ msgstr ""
#~ "Після оновлення інформації про пакунки, суттєвий пакунок '%s' не можливо "
#~ "знайти.\n"
#~ "Це вказує на серйозну помилку. Повідомте, будь-ласка, про цю помилку, "
#~ "використовуючи в Терміналі команду 'ubuntu-bug update-manager', та "
#~ "додайте до звіту файли з /var/log/dist-upgrade/."

#~ msgid ""
#~ "The system was unable to get the prerequisites for the upgrade. The "
#~ "upgrade will abort now and restore the original system state.\n"
#~ "\n"
#~ "Please report this as a bug using the command 'ubuntu-bug update-manager' "
#~ "in a terminal and include the files in /var/log/dist-upgrade/ in the bug "
#~ "report."
#~ msgstr ""
#~ "Системі не вдалося отримати необхідні дані оновлення. Зараз оновлення "
#~ "буде перервано, і система повернеться до попереднього стану.\n"
#~ "\n"
#~ "Повідомте, будь-ласка, про цю помилку, використовуючи в Терміналі команду "
#~ "'ubuntu-bug update-manager', та додайте до звіту файли з /var/log/dist-"
#~ "upgrade/."

#~ msgid ""
#~ "The support in Ubuntu 11.04 for your intel graphics hardware is limited "
#~ "and you may encounter problems after the upgrade. Do you want to continue "
#~ "with the upgrade?"
#~ msgstr ""
#~ "Підтримка вашого графічного обладнання intel обмежена, і ви можете "
#~ "зіткнутися з проблемами після оновлення. Ви хочете продовжити оновлення?"

#~ msgid ""
#~ "\n"
#~ "\n"
#~ "Please report this bug using the command 'ubuntu-bug update-manager' in a "
#~ "terminal and include the files in /var/log/dist-upgrade/ in the bug "
#~ "report.\n"
#~ "%s"
#~ msgstr ""
#~ "\n"
#~ "\n"
#~ "Повідомте, будь-ласка, про цю помилку, використовуючи в Терміналі команду "
#~ "'ubuntu-bug update-manager', та додайте до звіту файли з /var/log/dist-"
#~ "upgrade/.\n"
#~ "%s"

#~ msgid ""
#~ "Preparing the system for the upgrade failed. Please report this using the "
#~ "command 'ubuntu-bug update-manager' in a terminal and include the files "
#~ "in /var/log/dist-upgrade/ in the bug report."
#~ msgstr ""
#~ "Не вдалося підготувати систему для оновлення. Повідомте, будь-ласка, про "
#~ "це, використовуючи в Терміналі команду 'ubuntu-bug update-manager', та "
#~ "додайте до звіту файли з /var/log/dist-upgrade/."

#~ msgid "%.0f kB"
#~ msgstr "%.0f кБ"

#~ msgid "0 kB"
#~ msgstr "0 кБ"

#~ msgid "1 kB"
#~ msgstr "1 кБ"

#~ msgid "Your graphics hardware may not be fully supported in Ubuntu 11.04."
#~ msgstr ""
#~ "Ваше графічне обладнання може не повністю підтримуватись в Ubuntu 11.04."

#~ msgid ""
#~ "These software updates have been issued since this version of Ubuntu was "
#~ "released. If you don't want to install them now, choose \"Update Manager"
#~ "\" from Applications later."
#~ msgstr ""
#~ "Ці оновлення програм були випущені після того, як вийшла ця версія "
#~ "Ubuntu. Якщо ви не хочете зараз їх встановлювати, оберіть \"Менеджер "
#~ "оновлення\" з меню Програми пізніше."

#~ msgid ""
#~ "These software updates have been issued since this version of Ubuntu was "
#~ "released. If you don't want to install them now, choose \"Update Manager"
#~ "\" from the Administration Menu later."
#~ msgstr ""
#~ "Ці оновлення програм були випущені після того, як вийшла ця версія "
#~ "Ubuntu. Якщо ви не хочете зараз їх встановлювати, оберіть \"Менеджер "
#~ "оновлення\" з меню Адміністрування пізніше."

#~ msgid ""
#~ "Upgrading the repository information resulted in a invalid file. Please "
#~ "report this as a bug using the command 'ubuntu-bug update-manager' in a "
#~ "terminal."
#~ msgstr ""
#~ "Оновлення інформації репозиторію завершилося помилкою у файлі. Повідомте, "
#~ "будь-ласка, про цю помилку, використовуючи в Терміналі команду 'ubuntu-"
#~ "bug update-manager'."
